<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.555">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>kyyeee.github.io</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">

<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>

<script src="site_libs/plotly-binding-4.10.4/plotly.js"></script>

<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>

<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">

<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>

<link href="site_libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet">

<script src="site_libs/plotly-main-2.11.1/plotly-latest.min.js"></script>



<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">kyyeee.github.io</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link active" href="./index.html" aria-current="page"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#quarto" id="toc-quarto" class="nav-link active" data-scroll-target="#quarto">Quarto</a></li>
  <li><a href="#running-code" id="toc-running-code" class="nav-link" data-scroll-target="#running-code">Running Code</a></li>
  <li><a href="#load-library-and-datasets" id="toc-load-library-and-datasets" class="nav-link" data-scroll-target="#load-library-and-datasets">load library and datasets</a></li>
  <li><a href="#data-visualization" id="toc-data-visualization" class="nav-link" data-scroll-target="#data-visualization">Data visualization</a></li>
  <li><a href="#question-1" id="toc-question-1" class="nav-link" data-scroll-target="#question-1">Question #1:</a></li>
  <li><a href="#purrr-functions" id="toc-purrr-functions" class="nav-link" data-scroll-target="#purrr-functions">purrr functions:</a></li>
  <li><a href="#mixed-effects-modeling" id="toc-mixed-effects-modeling" class="nav-link" data-scroll-target="#mixed-effects-modeling">Mixed-Effects modeling</a>
  <ul class="collapse">
  <li><a href="#mixed-model" id="toc-mixed-model" class="nav-link" data-scroll-target="#mixed-model">mixed model</a></li>
  </ul></li>
  <li><a href="#new-data" id="toc-new-data" class="nav-link" data-scroll-target="#new-data">new data</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">kyyeee.github.io</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p>This is a Quarto website.</p>
<p>To learn more about Quarto websites visit <a href="https://quarto.org/docs/websites" class="uri">https://quarto.org/docs/websites</a>.</p>
<section id="quarto" class="level2">
<h2 class="anchored" data-anchor-id="quarto">Quarto</h2>
<p>Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <a href="https://quarto.org" class="uri">https://quarto.org</a>.</p>
</section>
<section id="running-code" class="level2">
<h2 class="anchored" data-anchor-id="running-code">Running Code</h2>
<p>When you click the <strong>Render</strong> button a document will be generated that includes both content and the output of embedded code. You can embed code like this:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># code for IPGTT_tidy_data_corrected</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="co"># opening libraries</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(plotly)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: ggplot2</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'plotly'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:ggplot2':

    last_plot</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:stats':

    filter</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:graphics':

    layout</code></pre>
</div>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.4     ✔ readr     2.1.5
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ lubridate 1.9.3     ✔ tibble    3.2.1
✔ purrr     1.0.2     ✔ tidyr     1.3.1</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks plotly::filter(), stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors</code></pre>
</div>
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(see)</span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggokabeito)</span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>here() starts at C:/Users/khopkins/Documents/kyyeee.github.io/IPGTTWebsite</code></pre>
</div>
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(conflicted)</span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(emmeans)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Welcome to emmeans.
Caution: You lose important information if you filter this package's results.
See '? untidy'</code></pre>
</div>
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>conflicted<span class="sc">::</span><span class="fu">conflicts_prefer</span>(dplyr<span class="sc">::</span>filter)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>[conflicted] Will prefer dplyr::filter over any other package.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="co"># create file</span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readxl)</span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a>IPGTT_tidy_data_corrected <span class="ot">&lt;-</span> <span class="fu">read_excel</span>(<span class="fu">here</span>(<span class="st">"IPGTT_tidy_data_corrected.xlsx"</span>))</span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a><span class="fu">View</span>(IPGTT_tidy_data_corrected)</span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-8"><a href="#cb16-8" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> IPGTT_tidy_data_corrected</span>
<span id="cb16-9"><a href="#cb16-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-10"><a href="#cb16-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-11"><a href="#cb16-11" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="sc">%&gt;%</span> <span class="fu">View</span>()</span>
<span id="cb16-12"><a href="#cb16-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-13"><a href="#cb16-13" aria-hidden="true" tabindex="-1"></a><span class="co"># previous variable names</span></span>
<span id="cb16-14"><a href="#cb16-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-15"><a href="#cb16-15" aria-hidden="true" tabindex="-1"></a><span class="co"># names(clean2_IPGTT_data)</span></span>
<span id="cb16-16"><a href="#cb16-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-17"><a href="#cb16-17" aria-hidden="true" tabindex="-1"></a><span class="co"># [1] "Mouse" "BW" "Test Week" "Genotype"</span></span>
<span id="cb16-18"><a href="#cb16-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-19"><a href="#cb16-19" aria-hidden="true" tabindex="-1"></a><span class="co"># [5] "Sex" "Time Point" "Glucose Reading"</span></span>
<span id="cb16-20"><a href="#cb16-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-21"><a href="#cb16-21" aria-hidden="true" tabindex="-1"></a><span class="co"># naming variables</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="co"># naming variables</span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(clean2_IPGTT_data) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"mouse"</span>, <span class="st">"body_weight"</span>, <span class="st">"test_week"</span>, <span class="st">"genotype"</span>, <span class="st">"sex"</span>, <span class="st">"minutes"</span>, <span class="st">"glucose"</span>)</span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a><span class="co"># replace "nan"s and "Mouse" in mouse variable</span></span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span> <span class="fu">mutate</span>(<span class="at">mouse =</span> <span class="fu">ifelse</span>(mouse <span class="sc">==</span> <span class="st">"nan"</span>, <span class="cn">NA</span>, mouse))</span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span> <span class="fu">mutate</span>(<span class="at">mouse =</span> <span class="fu">ifelse</span>(mouse <span class="sc">==</span> <span class="st">"Mouse"</span>, <span class="cn">NA</span>, mouse))</span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a><span class="co"># removing NA's from mouse variable</span></span>
<span id="cb17-9"><a href="#cb17-9" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span> <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(mouse))</span>
<span id="cb17-10"><a href="#cb17-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-11"><a href="#cb17-11" aria-hidden="true" tabindex="-1"></a><span class="co"># add unique ids for each mouse</span></span>
<span id="cb17-12"><a href="#cb17-12" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span> <span class="fu">mutate</span>(<span class="at">mouse_id =</span> <span class="fu">paste</span>(mouse, genotype, <span class="at">sep =</span> <span class="st">"_"</span>))</span>
<span id="cb17-13"><a href="#cb17-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-14"><a href="#cb17-14" aria-hidden="true" tabindex="-1"></a><span class="co"># ordering variables</span></span>
<span id="cb17-15"><a href="#cb17-15" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span> <span class="fu">select</span>(mouse_id, mouse, genotype, sex, test_week, minutes, <span class="fu">everything</span>())</span>
<span id="cb17-16"><a href="#cb17-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-17"><a href="#cb17-17" aria-hidden="true" tabindex="-1"></a><span class="co"># viewing variable structure</span></span>
<span id="cb17-18"><a href="#cb17-18" aria-hidden="true" tabindex="-1"></a><span class="fu">str</span>(clean2_IPGTT_data)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>tibble [1,968 × 8] (S3: tbl_df/tbl/data.frame)
 $ mouse_id   : chr [1:1968] "LP_KO-NOD" "LP_KO-NOD" "LP_KO-NOD" "LP_KO-NOD" ...
 $ mouse      : chr [1:1968] "LP" "LP" "LP" "LP" ...
 $ genotype   : chr [1:1968] "KO-NOD" "KO-NOD" "KO-NOD" "KO-NOD" ...
 $ sex        : chr [1:1968] "F" "F" "F" "F" ...
 $ test_week  : num [1:1968] 1 2 3 4 5 6 7 8 9 10 ...
 $ minutes    : chr [1:1968] "0 min" "0 min" "0 min" "0 min" ...
 $ body_weight: num [1:1968] 21.5 23.8 24.6 22.8 25.4 24.8 25.3 25.5 25 25.8 ...
 $ glucose    : num [1:1968] 114 114 133 96 158 155 155 161 137 252 ...</code></pre>
</div>
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="co"># changing "sex" and "genotype" variable to factor (change levels)</span></span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span> <span class="fu">mutate</span>(<span class="at">sex =</span> <span class="fu">as_factor</span>(sex))</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span></span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">genotype =</span> <span class="fu">factor</span>(genotype, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"WT-WT"</span>, <span class="st">"Het-NOD"</span>, <span class="st">"KO-NOD"</span>)))</span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-7"><a href="#cb19-7" aria-hidden="true" tabindex="-1"></a><span class="co"># removing "min" from minutes variable</span></span>
<span id="cb19-8"><a href="#cb19-8" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span> <span class="fu">mutate</span>(<span class="at">minutes =</span> <span class="fu">str_replace</span>(minutes, <span class="st">" min"</span>, <span class="st">""</span>))</span>
<span id="cb19-9"><a href="#cb19-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-10"><a href="#cb19-10" aria-hidden="true" tabindex="-1"></a><span class="co"># changing "minutes" variable to numeric</span></span>
<span id="cb19-11"><a href="#cb19-11" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span> <span class="fu">mutate</span>(<span class="at">minutes =</span> <span class="fu">as.numeric</span>(minutes))</span>
<span id="cb19-12"><a href="#cb19-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-13"><a href="#cb19-13" aria-hidden="true" tabindex="-1"></a><span class="co"># summary of variables</span></span>
<span id="cb19-14"><a href="#cb19-14" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(clean2_IPGTT_data)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>   mouse_id            mouse              genotype    sex        test_week     
 Length:1968        Length:1968        WT-WT  : 240   F:1968   Min.   : 1.000  
 Class :character   Class :character   Het-NOD: 288            1st Qu.: 3.000  
 Mode  :character   Mode  :character   KO-NOD :1440            Median : 6.000  
                                                               Mean   : 6.378  
                                                               3rd Qu.: 9.000  
                                                               Max.   :12.000  
                                                                               
    minutes        body_weight       glucose     
 Min.   :  0.00   Min.   :16.60   Min.   :  5.0  
 1st Qu.:  4.50   1st Qu.:20.07   1st Qu.: 68.0  
 Median : 22.50   Median :22.55   Median :118.0  
 Mean   : 37.83   Mean   :22.24   Mean   :168.5  
 3rd Qu.: 63.75   3rd Qu.:24.50   3rd Qu.:172.0  
 Max.   :120.00   Max.   :26.20   Max.   :750.0  
                  NA's   :1344    NA's   :1503   </code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="co"># creating data dictionary</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>data_dictionary <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">Variable =</span> <span class="fu">c</span>(</span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>    <span class="st">"mouse_id"</span>, <span class="st">"mouse"</span>, <span class="st">"genotype"</span>, <span class="st">"sex"</span>, <span class="st">"test_week"</span>, <span class="st">"minutes"</span>, <span class="st">"body_weight"</span>, <span class="st">"glucose"</span></span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">Type =</span> <span class="fu">c</span>(</span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Character"</span>, <span class="st">"Character"</span>, <span class="st">"Factor"</span>, <span class="st">"Factor"</span>, <span class="st">"Numeric"</span>, </span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Numeric"</span>, <span class="st">"Numeric"</span>, <span class="st">"Numeric"</span></span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">Description =</span> <span class="fu">c</span>(</span>
<span id="cb21-11"><a href="#cb21-11" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Unique identifier for each mouse"</span>,</span>
<span id="cb21-12"><a href="#cb21-12" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Where the the mouse was pierced or the given number to differentiate"</span>,</span>
<span id="cb21-13"><a href="#cb21-13" aria-hidden="true" tabindex="-1"></a>    <span class="st">"the genotype of the mouse"</span>,</span>
<span id="cb21-14"><a href="#cb21-14" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Gender of the mouse"</span>,</span>
<span id="cb21-15"><a href="#cb21-15" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The week of testing the results were taken"</span>,</span>
<span id="cb21-16"><a href="#cb21-16" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Minutes after glucose intake (minutes)"</span>,</span>
<span id="cb21-17"><a href="#cb21-17" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Weight of the mouse in grams (grams)"</span>,</span>
<span id="cb21-18"><a href="#cb21-18" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Amount of glucose in the mouse's bloodstream (mg/dL)"</span></span>
<span id="cb21-19"><a href="#cb21-19" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb21-20"><a href="#cb21-20" aria-hidden="true" tabindex="-1"></a>  <span class="at">PossibleValues =</span> <span class="fu">c</span>(</span>
<span id="cb21-21"><a href="#cb21-21" aria-hidden="true" tabindex="-1"></a>    <span class="st">"unique mouse string + genotype"</span>,</span>
<span id="cb21-22"><a href="#cb21-22" aria-hidden="true" tabindex="-1"></a>    <span class="st">"mouse string"</span>,</span>
<span id="cb21-23"><a href="#cb21-23" aria-hidden="true" tabindex="-1"></a>    <span class="st">"genotype "</span>,</span>
<span id="cb21-24"><a href="#cb21-24" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Male, Female, Male/Female"</span>,</span>
<span id="cb21-25"><a href="#cb21-25" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Positive numbers"</span>,</span>
<span id="cb21-26"><a href="#cb21-26" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Positive numbers"</span>,</span>
<span id="cb21-27"><a href="#cb21-27" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Positive numbers"</span>,</span>
<span id="cb21-28"><a href="#cb21-28" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Positive numbers"</span></span>
<span id="cb21-29"><a href="#cb21-29" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb21-30"><a href="#cb21-30" aria-hidden="true" tabindex="-1"></a>  <span class="at">Example =</span> <span class="fu">c</span>(</span>
<span id="cb21-31"><a href="#cb21-31" aria-hidden="true" tabindex="-1"></a>    <span class="st">"203_KO-NOD, RP_Het-NOD, ..."</span>,</span>
<span id="cb21-32"><a href="#cb21-32" aria-hidden="true" tabindex="-1"></a>    <span class="st">"LF, MP, 203, 205, ..."</span>,</span>
<span id="cb21-33"><a href="#cb21-33" aria-hidden="true" tabindex="-1"></a>    <span class="st">"KO-NOD, Het-NOD, WT-WT, ..."</span>,</span>
<span id="cb21-34"><a href="#cb21-34" aria-hidden="true" tabindex="-1"></a>    <span class="st">"'M', 'F'"</span>,</span>
<span id="cb21-35"><a href="#cb21-35" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1, 2, 3, 12, ..."</span>,</span>
<span id="cb21-36"><a href="#cb21-36" aria-hidden="true" tabindex="-1"></a>    <span class="st">"0, 60, 120, ..."</span>,</span>
<span id="cb21-37"><a href="#cb21-37" aria-hidden="true" tabindex="-1"></a>    <span class="st">"24.5, 22.9, ..."</span>,</span>
<span id="cb21-38"><a href="#cb21-38" aria-hidden="true" tabindex="-1"></a>    <span class="st">"67, 100, ..."</span></span>
<span id="cb21-39"><a href="#cb21-39" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb21-40"><a href="#cb21-40" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="load-library-and-datasets" class="level2">
<h2 class="anchored" data-anchor-id="load-library-and-datasets">load library and datasets</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a><span class="co"># load libraries</span></span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a>okabe_ito_palette <span class="ot">&lt;-</span> <span class="fu">rep</span>(grDevices<span class="sc">::</span><span class="fu">palette.colors</span>(<span class="at">palette =</span> <span class="st">"Okabe-Ito"</span>))</span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a><span class="co"># assign colors</span></span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a>genotype_colors <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"WT-WT"</span> <span class="ot">=</span> <span class="st">"#009E73"</span>, <span class="st">"Het-NOD"</span> <span class="ot">=</span> <span class="st">"#D55E00"</span>, <span class="st">"KO-NOD"</span> <span class="ot">=</span> <span class="st">"#56B4E9"</span>)</span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-9"><a href="#cb22-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-10"><a href="#cb22-10" aria-hidden="true" tabindex="-1"></a><span class="co">#assign data</span></span>
<span id="cb22-11"><a href="#cb22-11" aria-hidden="true" tabindex="-1"></a>mean_glucose_testweek <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span></span>
<span id="cb22-12"><a href="#cb22-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(test_week) <span class="sc">%&gt;%</span></span>
<span id="cb22-13"><a href="#cb22-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarize</span>(<span class="at">mean_glucose =</span> <span class="fu">mean</span>(glucose, <span class="at">na.rm =</span> <span class="cn">TRUE</span>))</span>
<span id="cb22-14"><a href="#cb22-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-15"><a href="#cb22-15" aria-hidden="true" tabindex="-1"></a>combined_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span></span>
<span id="cb22-16"><a href="#cb22-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(test_week, glucose, genotype)</span>
<span id="cb22-17"><a href="#cb22-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-18"><a href="#cb22-18" aria-hidden="true" tabindex="-1"></a>combined_data<span class="sc">$</span>test_week <span class="ot">&lt;-</span> <span class="fu">factor</span>(combined_data<span class="sc">$</span>test_week, <span class="at">levels =</span> <span class="fu">sort</span>(<span class="fu">unique</span>(combined_data<span class="sc">$</span>test_week)))</span>
<span id="cb22-19"><a href="#cb22-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-20"><a href="#cb22-20" aria-hidden="true" tabindex="-1"></a>avg_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span></span>
<span id="cb22-21"><a href="#cb22-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(test_week) <span class="sc">%&gt;%</span></span>
<span id="cb22-22"><a href="#cb22-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarize</span>(</span>
<span id="cb22-23"><a href="#cb22-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">avg_body_weight =</span> <span class="fu">mean</span>(body_weight, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb22-24"><a href="#cb22-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">avg_glucose =</span> <span class="fu">mean</span>(glucose, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb22-25"><a href="#cb22-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">.groups =</span> <span class="st">'drop'</span></span>
<span id="cb22-26"><a href="#cb22-26" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="data-visualization" class="level1">
<h1>Data visualization</h1>
</section>
<section id="question-1" class="level1">
<h1>Question #1:</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Question #2: How does genotpye affect average body weight change over each week</span></span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate means by genotype</span></span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a>mean_bodyweight_testweek_genotype <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span></span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(test_week, genotype) <span class="sc">%&gt;%</span></span>
<span id="cb23-5"><a href="#cb23-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarize</span>(<span class="at">mean_body_weight =</span> <span class="fu">mean</span>(body_weight, <span class="at">na.rm =</span> <span class="cn">TRUE</span>), <span class="at">.groups =</span> <span class="st">'drop'</span>)</span>
<span id="cb23-6"><a href="#cb23-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-7"><a href="#cb23-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate overall means</span></span>
<span id="cb23-8"><a href="#cb23-8" aria-hidden="true" tabindex="-1"></a>overall_mean_bodyweight_testweek <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span></span>
<span id="cb23-9"><a href="#cb23-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(test_week) <span class="sc">%&gt;%</span></span>
<span id="cb23-10"><a href="#cb23-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarize</span>(<span class="at">mean_body_weight =</span> <span class="fu">mean</span>(body_weight, <span class="at">na.rm =</span> <span class="cn">TRUE</span>), <span class="at">.groups =</span> <span class="st">'drop'</span>)</span>
<span id="cb23-11"><a href="#cb23-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-12"><a href="#cb23-12" aria-hidden="true" tabindex="-1"></a><span class="co"># plot data</span></span>
<span id="cb23-13"><a href="#cb23-13" aria-hidden="true" tabindex="-1"></a>o <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(clean2_IPGTT_data, <span class="fu">aes</span>(<span class="at">x =</span> test_week, <span class="at">y =</span> body_weight, <span class="at">color =</span> genotype)) <span class="sc">+</span></span>
<span id="cb23-14"><a href="#cb23-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">alpha =</span> <span class="fl">0.05</span>) <span class="sc">+</span> <span class="co"># Individual data points</span></span>
<span id="cb23-15"><a href="#cb23-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">data =</span> mean_bodyweight_testweek_genotype, </span>
<span id="cb23-16"><a href="#cb23-16" aria-hidden="true" tabindex="-1"></a>             <span class="fu">aes</span>(<span class="at">x =</span> test_week, <span class="at">y =</span> mean_body_weight), </span>
<span id="cb23-17"><a href="#cb23-17" aria-hidden="true" tabindex="-1"></a>             <span class="at">size =</span> <span class="dv">3</span>, <span class="at">shape =</span> <span class="dv">18</span>) <span class="sc">+</span> <span class="co"># Mean points for each genotype</span></span>
<span id="cb23-18"><a href="#cb23-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">data =</span> mean_bodyweight_testweek_genotype, </span>
<span id="cb23-19"><a href="#cb23-19" aria-hidden="true" tabindex="-1"></a>            <span class="fu">aes</span>(<span class="at">x =</span> test_week, <span class="at">y =</span> mean_body_weight, <span class="at">group =</span> genotype), </span>
<span id="cb23-20"><a href="#cb23-20" aria-hidden="true" tabindex="-1"></a>            <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span> <span class="co"># Connecting lines for each genotype</span></span>
<span id="cb23-21"><a href="#cb23-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">data =</span> overall_mean_bodyweight_testweek, </span>
<span id="cb23-22"><a href="#cb23-22" aria-hidden="true" tabindex="-1"></a>             <span class="fu">aes</span>(<span class="at">x =</span> test_week, <span class="at">y =</span> mean_body_weight), </span>
<span id="cb23-23"><a href="#cb23-23" aria-hidden="true" tabindex="-1"></a>             <span class="at">size =</span> <span class="dv">3</span>, <span class="at">shape =</span> <span class="dv">18</span>, <span class="at">color =</span> <span class="st">"black"</span>) <span class="sc">+</span> <span class="co"># Mean points for all data</span></span>
<span id="cb23-24"><a href="#cb23-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">data =</span> overall_mean_bodyweight_testweek, </span>
<span id="cb23-25"><a href="#cb23-25" aria-hidden="true" tabindex="-1"></a>            <span class="fu">aes</span>(<span class="at">x =</span> test_week, <span class="at">y =</span> mean_body_weight), </span>
<span id="cb23-26"><a href="#cb23-26" aria-hidden="true" tabindex="-1"></a>            <span class="at">size =</span> <span class="dv">1</span>, <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>) <span class="sc">+</span> <span class="co"># Mean line for all data</span></span>
<span id="cb23-27"><a href="#cb23-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"Scatter Plot of Body Weight Over Time by Genotype for Each Week"</span>, </span>
<span id="cb23-28"><a href="#cb23-28" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Test Week"</span>, </span>
<span id="cb23-29"><a href="#cb23-29" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Body Weight"</span>) <span class="sc">+</span></span>
<span id="cb23-30"><a href="#cb23-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb23-31"><a href="#cb23-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_okabe_ito</span>() <span class="sc">+</span></span>
<span id="cb23-32"><a href="#cb23-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.title =</span> <span class="fu">element_blank</span>(), <span class="at">legend.position =</span> <span class="st">"bottom"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Using `size` aesthetic for lines was deprecated in ggplot2 3.4.0.
ℹ Please use `linewidth` instead.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplotly</span>(o)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="plotly html-widget html-fill-item" id="htmlwidget-e35c5c77e617b12aaee3" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-e35c5c77e617b12aaee3">{"x":{"data":[{"x":[1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10],"y":[20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null,20.100000000000001,20,20.5,20.699999999999999,null,null,null,null,null,null,19,18.300000000000001,19.399999999999999,19.399999999999999,null,null,null,null,null,null],"text":["test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 20.1<br />genotype: WT-WT","test_week:  2<br />body_weight: 20.0<br />genotype: WT-WT","test_week:  3<br />body_weight: 20.5<br />genotype: WT-WT","test_week:  4<br />body_weight: 20.7<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT","test_week:  1<br />body_weight: 19.0<br />genotype: WT-WT","test_week:  2<br />body_weight: 18.3<br />genotype: WT-WT","test_week:  3<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  4<br />body_weight: 19.4<br />genotype: WT-WT","test_week:  5<br />body_weight:   NA<br />genotype: WT-WT","test_week:  6<br />body_weight:   NA<br />genotype: WT-WT","test_week:  7<br />body_weight:   NA<br />genotype: WT-WT","test_week:  8<br />body_weight:   NA<br />genotype: WT-WT","test_week:  9<br />body_weight:   NA<br />genotype: WT-WT","test_week: 10<br />body_weight:   NA<br />genotype: WT-WT"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(230,159,0,1)","opacity":0.050000000000000003,"size":5.6692913385826778,"symbol":"circle","line":{"width":1.8897637795275593,"color":"rgba(230,159,0,1)"}},"hoveron":"points","name":"WT-WT","legendgroup":"WT-WT","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12],"y":[23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null,23.100000000000001,23.199999999999999,23.199999999999999,22.899999999999999,24.399999999999999,24.100000000000001,24.5,24.699999999999999,25,24.5,24.5,24,21,21.899999999999999,20,19.5,21.5,22.899999999999999,22.600000000000001,23,20,18.300000000000001,16.600000000000001,null],"text":["test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD","test_week:  1<br />body_weight: 23.1<br />genotype: Het-NOD","test_week:  2<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  3<br />body_weight: 23.2<br />genotype: Het-NOD","test_week:  4<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  5<br />body_weight: 24.4<br />genotype: Het-NOD","test_week:  6<br />body_weight: 24.1<br />genotype: Het-NOD","test_week:  7<br />body_weight: 24.5<br />genotype: Het-NOD","test_week:  8<br />body_weight: 24.7<br />genotype: Het-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 11<br />body_weight: 24.5<br />genotype: Het-NOD","test_week: 12<br />body_weight: 24.0<br />genotype: Het-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: Het-NOD","test_week:  2<br />body_weight: 21.9<br />genotype: Het-NOD","test_week:  3<br />body_weight: 20.0<br />genotype: Het-NOD","test_week:  4<br />body_weight: 19.5<br />genotype: Het-NOD","test_week:  5<br />body_weight: 21.5<br />genotype: Het-NOD","test_week:  6<br />body_weight: 22.9<br />genotype: Het-NOD","test_week:  7<br />body_weight: 22.6<br />genotype: Het-NOD","test_week:  8<br />body_weight: 23.0<br />genotype: Het-NOD","test_week:  9<br />body_weight: 20.0<br />genotype: Het-NOD","test_week: 10<br />body_weight: 18.3<br />genotype: Het-NOD","test_week: 11<br />body_weight: 16.6<br />genotype: Het-NOD","test_week: 12<br />body_weight:   NA<br />genotype: Het-NOD"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(86,180,233,1)","opacity":0.050000000000000003,"size":5.6692913385826778,"symbol":"circle","line":{"width":1.8897637795275593,"color":"rgba(86,180,233,1)"}},"hoveron":"points","name":"Het-NOD","legendgroup":"Het-NOD","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12],"y":[21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null,21.5,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999,21.899999999999999,null,null,null,null,null,null,null,null,null,null,null,21.100000000000001,null,null,null,null,null,null,null,null,null,null,null,21.600000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,22.5,null,null,null,null,null,null,null,null,null,null,null,20,null,null,null,null,null,null,null,null,null,null,null,20.300000000000001,null,null,null,null,null,null,null,null,null,null,null,19.300000000000001,null,null,null,null,null,null,null,null,null,null,null,21,null,null,null,null,null,null,null,null,null,null,null],"text":["test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.5<br />genotype: KO-NOD","test_week:  2<br />body_weight: 23.8<br />genotype: KO-NOD","test_week:  3<br />body_weight: 24.6<br />genotype: KO-NOD","test_week:  4<br />body_weight: 22.8<br />genotype: KO-NOD","test_week:  5<br />body_weight: 25.4<br />genotype: KO-NOD","test_week:  6<br />body_weight: 24.8<br />genotype: KO-NOD","test_week:  7<br />body_weight: 25.3<br />genotype: KO-NOD","test_week:  8<br />body_weight: 25.5<br />genotype: KO-NOD","test_week:  9<br />body_weight: 25.0<br />genotype: KO-NOD","test_week: 10<br />body_weight: 25.8<br />genotype: KO-NOD","test_week: 11<br />body_weight: 26.2<br />genotype: KO-NOD","test_week: 12<br />body_weight: 26.2<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.9<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.1<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.6<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 22.5<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 20.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 19.3<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  1<br />body_weight: 21.0<br />genotype: KO-NOD","test_week:  2<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  3<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  4<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  5<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  6<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  7<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  8<br />body_weight:   NA<br />genotype: KO-NOD","test_week:  9<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 10<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 11<br />body_weight:   NA<br />genotype: KO-NOD","test_week: 12<br />body_weight:   NA<br />genotype: KO-NOD"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.050000000000000003,"size":5.6692913385826778,"symbol":"circle","line":{"width":1.8897637795275593,"color":"rgba(0,158,115,1)"}},"hoveron":"points","name":"KO-NOD","legendgroup":"KO-NOD","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1,2,3,4,5,6,7,8,9,10],"y":[19.550000000000001,19.149999999999999,19.949999999999999,20.049999999999997,null,null,null,null,null,null],"text":["test_week:  1<br />mean_body_weight: 19.55<br />genotype: WT-WT","test_week:  2<br />mean_body_weight: 19.15<br />genotype: WT-WT","test_week:  3<br />mean_body_weight: 19.95<br />genotype: WT-WT","test_week:  4<br />mean_body_weight: 20.05<br />genotype: WT-WT","test_week:  5<br />mean_body_weight:   NaN<br />genotype: WT-WT","test_week:  6<br />mean_body_weight:   NaN<br />genotype: WT-WT","test_week:  7<br />mean_body_weight:   NaN<br />genotype: WT-WT","test_week:  8<br />mean_body_weight:   NaN<br />genotype: WT-WT","test_week:  9<br />mean_body_weight:   NaN<br />genotype: WT-WT","test_week: 10<br />mean_body_weight:   NaN<br />genotype: WT-WT"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(230,159,0,1)","opacity":1,"size":11.338582677165356,"symbol":"diamond","line":{"width":1.8897637795275593,"color":"rgba(230,159,0,1)"}},"hoveron":"points","name":"WT-WT","legendgroup":"WT-WT","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1,2,3,4,5,6,7,8,9,10,11,12],"y":[22.050000000000001,22.549999999999997,21.600000000000001,21.199999999999999,22.949999999999999,23.5,23.550000000000001,23.850000000000001,22.5,21.399999999999999,20.550000000000001,24],"text":["test_week:  1<br />mean_body_weight: 22.05<br />genotype: Het-NOD","test_week:  2<br />mean_body_weight: 22.55<br />genotype: Het-NOD","test_week:  3<br />mean_body_weight: 21.60<br />genotype: Het-NOD","test_week:  4<br />mean_body_weight: 21.20<br />genotype: Het-NOD","test_week:  5<br />mean_body_weight: 22.95<br />genotype: Het-NOD","test_week:  6<br />mean_body_weight: 23.50<br />genotype: Het-NOD","test_week:  7<br />mean_body_weight: 23.55<br />genotype: Het-NOD","test_week:  8<br />mean_body_weight: 23.85<br />genotype: Het-NOD","test_week:  9<br />mean_body_weight: 22.50<br />genotype: Het-NOD","test_week: 10<br />mean_body_weight: 21.40<br />genotype: Het-NOD","test_week: 11<br />mean_body_weight: 20.55<br />genotype: Het-NOD","test_week: 12<br />mean_body_weight: 24.00<br />genotype: Het-NOD"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(86,180,233,1)","opacity":1,"size":11.338582677165356,"symbol":"diamond","line":{"width":1.8897637795275593,"color":"rgba(86,180,233,1)"}},"hoveron":"points","name":"Het-NOD","legendgroup":"Het-NOD","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1,2,3,4,5,6,7,8,9,10,11,12],"y":[20.850000000000001,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999],"text":["test_week:  1<br />mean_body_weight: 20.85<br />genotype: KO-NOD","test_week:  2<br />mean_body_weight: 23.80<br />genotype: KO-NOD","test_week:  3<br />mean_body_weight: 24.60<br />genotype: KO-NOD","test_week:  4<br />mean_body_weight: 22.80<br />genotype: KO-NOD","test_week:  5<br />mean_body_weight: 25.40<br />genotype: KO-NOD","test_week:  6<br />mean_body_weight: 24.80<br />genotype: KO-NOD","test_week:  7<br />mean_body_weight: 25.30<br />genotype: KO-NOD","test_week:  8<br />mean_body_weight: 25.50<br />genotype: KO-NOD","test_week:  9<br />mean_body_weight: 25.00<br />genotype: KO-NOD","test_week: 10<br />mean_body_weight: 25.80<br />genotype: KO-NOD","test_week: 11<br />mean_body_weight: 26.20<br />genotype: KO-NOD","test_week: 12<br />mean_body_weight: 26.20<br />genotype: KO-NOD"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":1,"size":11.338582677165356,"symbol":"diamond","line":{"width":1.8897637795275593,"color":"rgba(0,158,115,1)"}},"hoveron":"points","name":"KO-NOD","legendgroup":"KO-NOD","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1,2,3,4,5,6,7,8,9,10],"y":[19.550000000000001,19.149999999999999,19.949999999999999,20.049999999999997,null,null,null,null,null,null],"text":["test_week:  1<br />mean_body_weight: 19.55<br />genotype: WT-WT<br />genotype: WT-WT","test_week:  2<br />mean_body_weight: 19.15<br />genotype: WT-WT<br />genotype: WT-WT","test_week:  3<br />mean_body_weight: 19.95<br />genotype: WT-WT<br />genotype: WT-WT","test_week:  4<br />mean_body_weight: 20.05<br />genotype: WT-WT<br />genotype: WT-WT","test_week:  5<br />mean_body_weight:   NaN<br />genotype: WT-WT<br />genotype: WT-WT","test_week:  6<br />mean_body_weight:   NaN<br />genotype: WT-WT<br />genotype: WT-WT","test_week:  7<br />mean_body_weight:   NaN<br />genotype: WT-WT<br />genotype: WT-WT","test_week:  8<br />mean_body_weight:   NaN<br />genotype: WT-WT<br />genotype: WT-WT","test_week:  9<br />mean_body_weight:   NaN<br />genotype: WT-WT<br />genotype: WT-WT","test_week: 10<br />mean_body_weight:   NaN<br />genotype: WT-WT<br />genotype: WT-WT"],"type":"scatter","mode":"lines","line":{"width":3.7795275590551185,"color":"rgba(230,159,0,1)","dash":"solid"},"hoveron":"points","name":"WT-WT","legendgroup":"WT-WT","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1,2,3,4,5,6,7,8,9,10,11,12],"y":[22.050000000000001,22.549999999999997,21.600000000000001,21.199999999999999,22.949999999999999,23.5,23.550000000000001,23.850000000000001,22.5,21.399999999999999,20.550000000000001,24],"text":["test_week:  1<br />mean_body_weight: 22.05<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week:  2<br />mean_body_weight: 22.55<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week:  3<br />mean_body_weight: 21.60<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week:  4<br />mean_body_weight: 21.20<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week:  5<br />mean_body_weight: 22.95<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week:  6<br />mean_body_weight: 23.50<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week:  7<br />mean_body_weight: 23.55<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week:  8<br />mean_body_weight: 23.85<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week:  9<br />mean_body_weight: 22.50<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week: 10<br />mean_body_weight: 21.40<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week: 11<br />mean_body_weight: 20.55<br />genotype: Het-NOD<br />genotype: Het-NOD","test_week: 12<br />mean_body_weight: 24.00<br />genotype: Het-NOD<br />genotype: Het-NOD"],"type":"scatter","mode":"lines","line":{"width":3.7795275590551185,"color":"rgba(86,180,233,1)","dash":"solid"},"hoveron":"points","name":"Het-NOD","legendgroup":"Het-NOD","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1,2,3,4,5,6,7,8,9,10,11,12],"y":[20.850000000000001,23.800000000000001,24.600000000000001,22.800000000000001,25.399999999999999,24.800000000000001,25.300000000000001,25.5,25,25.800000000000001,26.199999999999999,26.199999999999999],"text":["test_week:  1<br />mean_body_weight: 20.85<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week:  2<br />mean_body_weight: 23.80<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week:  3<br />mean_body_weight: 24.60<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week:  4<br />mean_body_weight: 22.80<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week:  5<br />mean_body_weight: 25.40<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week:  6<br />mean_body_weight: 24.80<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week:  7<br />mean_body_weight: 25.30<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week:  8<br />mean_body_weight: 25.50<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week:  9<br />mean_body_weight: 25.00<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week: 10<br />mean_body_weight: 25.80<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week: 11<br />mean_body_weight: 26.20<br />genotype: KO-NOD<br />genotype: KO-NOD","test_week: 12<br />mean_body_weight: 26.20<br />genotype: KO-NOD<br />genotype: KO-NOD"],"type":"scatter","mode":"lines","line":{"width":3.7795275590551185,"color":"rgba(0,158,115,1)","dash":"solid"},"hoveron":"points","name":"KO-NOD","legendgroup":"KO-NOD","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1,2,3,4,5,6,7,8,9,10,11,12],"y":[20.835714285714285,21.440000000000001,21.539999999999999,21.059999999999999,23.766666666666666,23.933333333333334,24.133333333333333,24.399999999999999,23.333333333333332,22.866666666666667,22.433333333333334,25.100000000000001],"text":["test_week:  1<br />mean_body_weight: 20.83571<br />genotype: black","test_week:  2<br />mean_body_weight: 21.44000<br />genotype: black","test_week:  3<br />mean_body_weight: 21.54000<br />genotype: black","test_week:  4<br />mean_body_weight: 21.06000<br />genotype: black","test_week:  5<br />mean_body_weight: 23.76667<br />genotype: black","test_week:  6<br />mean_body_weight: 23.93333<br />genotype: black","test_week:  7<br />mean_body_weight: 24.13333<br />genotype: black","test_week:  8<br />mean_body_weight: 24.40000<br />genotype: black","test_week:  9<br />mean_body_weight: 23.33333<br />genotype: black","test_week: 10<br />mean_body_weight: 22.86667<br />genotype: black","test_week: 11<br />mean_body_weight: 22.43333<br />genotype: black","test_week: 12<br />mean_body_weight: 25.10000<br />genotype: black"],"type":"scatter","mode":"markers+lines","marker":{"autocolorscale":false,"color":"rgba(0,0,0,1)","opacity":1,"size":11.338582677165356,"symbol":"diamond","line":{"width":1.8897637795275593,"color":"rgba(0,0,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","line":{"width":3.7795275590551185,"color":"rgba(0,0,0,1)","dash":"dash"},"frame":null}],"layout":{"margin":{"t":43.762557077625573,"r":7.3059360730593621,"b":40.182648401826491,"l":37.260273972602747},"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"title":{"text":"Scatter Plot of Body Weight Over Time by Genotype for Each Week","font":{"color":"rgba(0,0,0,1)","family":"","size":17.534246575342465},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.44999999999999996,12.550000000000001],"tickmode":"array","ticktext":["2.5","5.0","7.5","10.0","12.5"],"tickvals":[2.5,5,7.5,10,12.5],"categoryorder":"array","categoryarray":["2.5","5.0","7.5","10.0","12.5"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"y","title":{"text":"Test Week","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[16.120000000000001,26.68],"tickmode":"array","ticktext":["18","20","22","24","26"],"tickvals":[18,20,22,24,26],"categoryorder":"array","categoryarray":["18","20","22","24","26"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"x","title":{"text":"Body Weight","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"title":{"text":"genotype","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5ee46c8e4f95":{"x":{},"y":{},"colour":{},"type":"scatter"},"5ee4246846ea":{"x":{},"y":{},"colour":{}},"5ee46c63452d":{"x":{},"y":{},"colour":{}},"5ee44ea16882":{"x":{},"y":{},"colour":{}},"5ee4631319ee":{"x":{},"y":{},"colour":{}}},"cur_data":"5ee46c8e4f95","visdat":{"5ee46c8e4f95":["function (y) ","x"],"5ee4246846ea":["function (y) ","x"],"5ee46c63452d":["function (y) ","x"],"5ee44ea16882":["function (y) ","x"],"5ee4631319ee":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="purrr-functions" class="level1">
<h1>purrr functions:</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Load libraries</span></span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(gridExtra)</span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggokabeito)</span>
<span id="cb26-6"><a href="#cb26-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggtext)</span>
<span id="cb26-7"><a href="#cb26-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(showtext)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: sysfonts</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: showtextdb</code></pre>
</div>
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Add Source font</span></span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a>test <span class="ot">&lt;-</span> <span class="fu">font_add_google</span>(<span class="st">"Source Code Pro"</span>, <span class="at">family =</span> <span class="st">"source"</span>)</span>
<span id="cb29-3"><a href="#cb29-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-4"><a href="#cb29-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Enable Font Usage</span></span>
<span id="cb29-5"><a href="#cb29-5" aria-hidden="true" tabindex="-1"></a><span class="fu">showtext_auto</span>()</span>
<span id="cb29-6"><a href="#cb29-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-7"><a href="#cb29-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Define plotting function</span></span>
<span id="cb29-8"><a href="#cb29-8" aria-hidden="true" tabindex="-1"></a>plot_glucose_trajectory_grouped <span class="ot">&lt;-</span> <span class="cf">function</span>(clean2_IPGTT_data, week_range, genotype_pair, genotype_colors) {</span>
<span id="cb29-9"><a href="#cb29-9" aria-hidden="true" tabindex="-1"></a>  filtered_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span></span>
<span id="cb29-10"><a href="#cb29-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(test_week <span class="sc">%in%</span> week_range, genotype <span class="sc">%in%</span> genotype_pair, <span class="sc">!</span><span class="fu">is.na</span>(glucose))</span>
<span id="cb29-11"><a href="#cb29-11" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb29-12"><a href="#cb29-12" aria-hidden="true" tabindex="-1"></a>  median_gdata <span class="ot">&lt;-</span> filtered_data <span class="sc">%&gt;%</span></span>
<span id="cb29-13"><a href="#cb29-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">group_by</span>(test_week, minutes, genotype) <span class="sc">%&gt;%</span></span>
<span id="cb29-14"><a href="#cb29-14" aria-hidden="true" tabindex="-1"></a>    <span class="fu">summarize</span>(<span class="at">median_glucose =</span> <span class="fu">median</span>(glucose, <span class="at">na.rm =</span> <span class="cn">TRUE</span>), <span class="at">.groups =</span> <span class="st">'drop'</span>)</span>
<span id="cb29-15"><a href="#cb29-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-16"><a href="#cb29-16" aria-hidden="true" tabindex="-1"></a>  <span class="co"># colored title text</span></span>
<span id="cb29-17"><a href="#cb29-17" aria-hidden="true" tabindex="-1"></a>  colored_genotype1 <span class="ot">&lt;-</span> <span class="fu">paste0</span>(<span class="st">"&lt;span style='color:"</span>, genotype_colors[genotype_pair[<span class="dv">1</span>]], <span class="st">"'&gt;"</span>, genotype_pair[<span class="dv">1</span>], <span class="st">"&lt;/span&gt;"</span>)</span>
<span id="cb29-18"><a href="#cb29-18" aria-hidden="true" tabindex="-1"></a>  colored_genotype2 <span class="ot">&lt;-</span> <span class="fu">paste0</span>(<span class="st">"&lt;span style='color:"</span>, genotype_colors[genotype_pair[<span class="dv">2</span>]], <span class="st">"'&gt;"</span>, genotype_pair[<span class="dv">2</span>], <span class="st">"&lt;/span&gt;"</span>)</span>
<span id="cb29-19"><a href="#cb29-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-20"><a href="#cb29-20" aria-hidden="true" tabindex="-1"></a><span class="co"># plot data</span></span>
<span id="cb29-21"><a href="#cb29-21" aria-hidden="true" tabindex="-1"></a>  m <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> filtered_data, <span class="fu">aes</span>(<span class="at">x =</span> minutes, <span class="at">y =</span> glucose, <span class="at">color =</span> genotype)) <span class="sc">+</span></span>
<span id="cb29-22"><a href="#cb29-22" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_point</span>(<span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb29-23"><a href="#cb29-23" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_line</span>(<span class="fu">aes</span>(<span class="at">group =</span> mouse_id), <span class="at">alpha =</span> <span class="fl">0.3</span>) <span class="sc">+</span></span>
<span id="cb29-24"><a href="#cb29-24" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_line</span>(<span class="at">data =</span> median_gdata, <span class="fu">aes</span>(<span class="at">x =</span> minutes, <span class="at">y =</span> median_glucose, <span class="at">color =</span> genotype, <span class="at">group =</span> <span class="fu">interaction</span>(genotype, test_week)), <span class="at">size =</span> <span class="fl">1.2</span>) <span class="sc">+</span></span>
<span id="cb29-25"><a href="#cb29-25" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">title =</span> <span class="fu">paste</span>(<span class="st">"Glucose Trajectories of"</span>, colored_genotype1, <span class="st">"vs"</span>, colored_genotype2, <span class="st">"for Weeks"</span>, <span class="fu">min</span>(week_range), <span class="st">"-"</span>, <span class="fu">max</span>(week_range)),</span>
<span id="cb29-26"><a href="#cb29-26" aria-hidden="true" tabindex="-1"></a>         <span class="at">subtitle =</span> <span class="fu">paste</span>(<span class="st">"for Weeks"</span>, <span class="fu">min</span>(week_range), <span class="st">"-"</span>, <span class="fu">max</span>(week_range)),</span>
<span id="cb29-27"><a href="#cb29-27" aria-hidden="true" tabindex="-1"></a>         <span class="at">x =</span> <span class="st">"Time (minutes)"</span>, <span class="at">y =</span> <span class="st">"Glucose Reading (mg/dL)"</span>) <span class="sc">+</span></span>
<span id="cb29-28"><a href="#cb29-28" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb29-29"><a href="#cb29-29" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_x_continuous</span>(<span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">30</span>, <span class="dv">60</span>, <span class="dv">90</span>, <span class="dv">120</span>)) <span class="sc">+</span></span>
<span id="cb29-30"><a href="#cb29-30" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_color_manual</span>(<span class="at">values =</span> genotype_colors) <span class="sc">+</span></span>
<span id="cb29-31"><a href="#cb29-31" aria-hidden="true" tabindex="-1"></a>    <span class="fu">facet_wrap</span>(<span class="sc">~</span> test_week, <span class="at">ncol =</span> <span class="dv">4</span>) <span class="sc">+</span></span>
<span id="cb29-32"><a href="#cb29-32" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(</span>
<span id="cb29-33"><a href="#cb29-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">family =</span> <span class="st">"source"</span>),</span>
<span id="cb29-34"><a href="#cb29-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="dv">0</span>, <span class="at">family =</span> <span class="st">"source"</span>),</span>
<span id="cb29-35"><a href="#cb29-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="dv">0</span>, <span class="at">family =</span> <span class="st">"source"</span>),</span>
<span id="cb29-36"><a href="#cb29-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text =</span> <span class="fu">element_text</span>(<span class="at">family =</span> <span class="st">"source"</span>),</span>
<span id="cb29-37"><a href="#cb29-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">family =</span> <span class="st">"source"</span>),</span>
<span id="cb29-38"><a href="#cb29-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text =</span> <span class="fu">element_text</span>(<span class="at">family =</span> <span class="st">"source"</span>),</span>
<span id="cb29-39"><a href="#cb29-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">family =</span> <span class="st">"source"</span>),</span>
<span id="cb29-40"><a href="#cb29-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_text</span>(<span class="at">family =</span> <span class="st">"source"</span>),</span>
<span id="cb29-41"><a href="#cb29-41" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb29-42"><a href="#cb29-42" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb29-43"><a href="#cb29-43" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplotly</span>(m)</span>
<span id="cb29-44"><a href="#cb29-44" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb29-45"><a href="#cb29-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-46"><a href="#cb29-46" aria-hidden="true" tabindex="-1"></a><span class="co"># Define variables</span></span>
<span id="cb29-47"><a href="#cb29-47" aria-hidden="true" tabindex="-1"></a>week_ranges <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>, <span class="dv">5</span><span class="sc">:</span><span class="dv">8</span>, <span class="dv">9</span><span class="sc">:</span><span class="dv">12</span>)</span>
<span id="cb29-48"><a href="#cb29-48" aria-hidden="true" tabindex="-1"></a>genotype_pairs <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="fu">c</span>(<span class="st">"WT-WT"</span>, <span class="st">"Het-NOD"</span>), <span class="fu">c</span>(<span class="st">"Het-NOD"</span>, <span class="st">"KO-NOD"</span>), <span class="fu">c</span>(<span class="st">"WT-WT"</span>, <span class="st">"KO-NOD"</span>))</span>
<span id="cb29-49"><a href="#cb29-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-50"><a href="#cb29-50" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-51"><a href="#cb29-51" aria-hidden="true" tabindex="-1"></a><span class="co"># plots for combinations of week range and genotype pair</span></span>
<span id="cb29-52"><a href="#cb29-52" aria-hidden="true" tabindex="-1"></a>plots <span class="ot">&lt;-</span> <span class="fu">map</span>(week_ranges, <span class="cf">function</span>(week_range) {</span>
<span id="cb29-53"><a href="#cb29-53" aria-hidden="true" tabindex="-1"></a>  <span class="fu">map</span>(genotype_pairs, <span class="cf">function</span>(genotype_pair) {</span>
<span id="cb29-54"><a href="#cb29-54" aria-hidden="true" tabindex="-1"></a>    <span class="fu">plot_glucose_trajectory_grouped</span>(clean2_IPGTT_data, week_range, genotype_pair, genotype_colors)</span>
<span id="cb29-55"><a href="#cb29-55" aria-hidden="true" tabindex="-1"></a>  })</span>
<span id="cb29-56"><a href="#cb29-56" aria-hidden="true" tabindex="-1"></a>})</span>
<span id="cb29-57"><a href="#cb29-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-58"><a href="#cb29-58" aria-hidden="true" tabindex="-1"></a><span class="co"># Flatten plots</span></span>
<span id="cb29-59"><a href="#cb29-59" aria-hidden="true" tabindex="-1"></a>plots <span class="ot">&lt;-</span> <span class="fu">flatten</span>(plots)</span>
<span id="cb29-60"><a href="#cb29-60" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-61"><a href="#cb29-61" aria-hidden="true" tabindex="-1"></a><span class="co"># Print "#" created plots</span></span>
<span id="cb29-62"><a href="#cb29-62" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">paste</span>(<span class="st">"Number of plots created:"</span>, <span class="fu">length</span>(plots)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] "Number of plots created: 9"</code></pre>
</div>
<div class="sourceCode cell-code" id="cb31"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Display plots</span></span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (plot <span class="cf">in</span> plots) {</span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">print</span>(plot)</span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a>}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="mixed-effects-modeling" class="level1">
<h1>Mixed-Effects modeling</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Load libraries</span></span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readr)</span>
<span id="cb32-3"><a href="#cb32-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span>
<span id="cb32-4"><a href="#cb32-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(lme4)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: Matrix</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'Matrix'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:tidyr':

    expand, pack, unpack</code></pre>
</div>
<div class="sourceCode cell-code" id="cb36"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb36-2"><a href="#cb36-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggeffects)</span>
<span id="cb36-3"><a href="#cb36-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-4"><a href="#cb36-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-5"><a href="#cb36-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Filter first week</span></span>
<span id="cb36-6"><a href="#cb36-6" aria-hidden="true" tabindex="-1"></a>week1_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span></span>
<span id="cb36-7"><a href="#cb36-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(test_week <span class="sc">==</span> <span class="dv">1</span>)</span>
<span id="cb36-8"><a href="#cb36-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-9"><a href="#cb36-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Fit model</span></span>
<span id="cb36-10"><a href="#cb36-10" aria-hidden="true" tabindex="-1"></a>model <span class="ot">&lt;-</span> <span class="fu">lm</span>(glucose <span class="sc">~</span> minutes <span class="sc">+</span> genotype, <span class="at">data =</span> week1_data)</span>
<span id="cb36-11"><a href="#cb36-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-12"><a href="#cb36-12" aria-hidden="true" tabindex="-1"></a><span class="co"># summary</span></span>
<span id="cb36-13"><a href="#cb36-13" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(model)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
lm(formula = glucose ~ minutes + genotype, data = week1_data)

Residuals:
    Min      1Q  Median      3Q     Max 
-61.037 -22.882   0.441  15.796  84.763 

Coefficients:
                  Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)      215.03733    6.89130  31.204  &lt; 2e-16 ***
minutes           -0.53555    0.06784  -7.895 1.74e-12 ***
genotypeHet-NOD  -75.18570   11.68251  -6.436 2.81e-09 ***
genotypeKO-NOD  -101.80024    7.18081 -14.177  &lt; 2e-16 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 29.11 on 117 degrees of freedom
  (47 observations deleted due to missingness)
Multiple R-squared:  0.693, Adjusted R-squared:  0.6851 
F-statistic: 88.02 on 3 and 117 DF,  p-value: &lt; 2.2e-16</code></pre>
</div>
<div class="sourceCode cell-code" id="cb38"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a><span class="co"># plot model</span></span>
<span id="cb38-2"><a href="#cb38-2" aria-hidden="true" tabindex="-1"></a>a <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(week1_data, <span class="fu">aes</span>(<span class="at">x =</span> minutes, <span class="at">y =</span> glucose, <span class="at">color =</span> genotype)) <span class="sc">+</span></span>
<span id="cb38-3"><a href="#cb38-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb38-4"><a href="#cb38-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_smooth</span>(<span class="at">method =</span> <span class="st">"lm"</span>, <span class="at">se =</span> <span class="cn">FALSE</span>) <span class="sc">+</span></span>
<span id="cb38-5"><a href="#cb38-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="fu">paste</span>(</span>
<span id="cb38-6"><a href="#cb38-6" aria-hidden="true" tabindex="-1"></a>      <span class="st">"&lt;span style='color:"</span>, genotype_colors[<span class="st">"WT-WT"</span>], <span class="st">"'&gt;WT-WT&lt;/span&gt; vs "</span>,</span>
<span id="cb38-7"><a href="#cb38-7" aria-hidden="true" tabindex="-1"></a>      <span class="st">"&lt;span style='color:"</span>, genotype_colors[<span class="st">"Het-NOD"</span>], <span class="st">"'&gt;Het-NOD&lt;/span&gt; vs "</span>,</span>
<span id="cb38-8"><a href="#cb38-8" aria-hidden="true" tabindex="-1"></a>      <span class="st">"&lt;span style='color:"</span>, genotype_colors[<span class="st">"KO-NOD"</span>], <span class="st">"'&gt;KO-NOD&lt;/span&gt;"</span>, <span class="st">"Blood Glucose Levels in the First Week"</span>),</span>
<span id="cb38-9"><a href="#cb38-9" aria-hidden="true" tabindex="-1"></a>      <span class="at">subtitle =</span> <span class="st">"Blood Glucose Levels in the First Week"</span>,</span>
<span id="cb38-10"><a href="#cb38-10" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Test Minutes"</span>,</span>
<span id="cb38-11"><a href="#cb38-11" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Blood Glucose"</span>,</span>
<span id="cb38-12"><a href="#cb38-12" aria-hidden="true" tabindex="-1"></a>       <span class="at">color =</span> <span class="st">"Genotype"</span>) <span class="sc">+</span></span>
<span id="cb38-13"><a href="#cb38-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb38-14"><a href="#cb38-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">30</span>, <span class="dv">60</span>, <span class="dv">90</span>, <span class="dv">120</span>)) <span class="sc">+</span></span>
<span id="cb38-15"><a href="#cb38-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> genotype_colors) <span class="sc">+</span></span>
<span id="cb38-16"><a href="#cb38-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb38-17"><a href="#cb38-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb38-18"><a href="#cb38-18" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb38-19"><a href="#cb38-19" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(a)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>`geom_smooth()` using formula = 'y ~ x'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Removed 47 rows containing non-finite outside the scale range
(`stat_smooth()`).</code></pre>
</div>
<div class="cell-output-display">
<div class="plotly html-widget html-fill-item" id="htmlwidget-aa641b8e53be195b8210" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-aa641b8e53be195b8210">{"x":{"data":[{"x":[0,0,1,1,3,3,5,5,10,10,15,15,30,30,45,45,60,60,75,75,90,90,120,120],"y":[187,154,191,172,199,173,217,228,256,237,262,195,219,190,null,null,172,192,null,null,180,202,162,155],"text":["minutes:   0<br />glucose: 187<br />genotype: WT-WT","minutes:   0<br />glucose: 154<br />genotype: WT-WT","minutes:   1<br />glucose: 191<br />genotype: WT-WT","minutes:   1<br />glucose: 172<br />genotype: WT-WT","minutes:   3<br />glucose: 199<br />genotype: WT-WT","minutes:   3<br />glucose: 173<br />genotype: WT-WT","minutes:   5<br />glucose: 217<br />genotype: WT-WT","minutes:   5<br />glucose: 228<br />genotype: WT-WT","minutes:  10<br />glucose: 256<br />genotype: WT-WT","minutes:  10<br />glucose: 237<br />genotype: WT-WT","minutes:  15<br />glucose: 262<br />genotype: WT-WT","minutes:  15<br />glucose: 195<br />genotype: WT-WT","minutes:  30<br />glucose: 219<br />genotype: WT-WT","minutes:  30<br />glucose: 190<br />genotype: WT-WT","minutes:  45<br />glucose:  NA<br />genotype: WT-WT","minutes:  45<br />glucose:  NA<br />genotype: WT-WT","minutes:  60<br />glucose: 172<br />genotype: WT-WT","minutes:  60<br />glucose: 192<br />genotype: WT-WT","minutes:  75<br />glucose:  NA<br />genotype: WT-WT","minutes:  75<br />glucose:  NA<br />genotype: WT-WT","minutes:  90<br />glucose: 180<br />genotype: WT-WT","minutes:  90<br />glucose: 202<br />genotype: WT-WT","minutes: 120<br />glucose: 162<br />genotype: WT-WT","minutes: 120<br />glucose: 155<br />genotype: WT-WT"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.5,"size":5.6692913385826778,"symbol":"circle","line":{"width":1.8897637795275593,"color":"rgba(0,158,115,1)"}},"hoveron":"points","name":"WT-WT","legendgroup":"WT-WT","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,0,1,1,3,3,5,5,10,10,15,15,30,30,45,45,60,60,75,75,90,90,120,120],"y":[115,112,null,null,null,null,null,null,null,null,161,155,157,90,null,null,131,63,null,null,114,null,null,null],"text":["minutes:   0<br />glucose: 115<br />genotype: Het-NOD","minutes:   0<br />glucose: 112<br />genotype: Het-NOD","minutes:   1<br />glucose:  NA<br />genotype: Het-NOD","minutes:   1<br />glucose:  NA<br />genotype: Het-NOD","minutes:   3<br />glucose:  NA<br />genotype: Het-NOD","minutes:   3<br />glucose:  NA<br />genotype: Het-NOD","minutes:   5<br />glucose:  NA<br />genotype: Het-NOD","minutes:   5<br />glucose:  NA<br />genotype: Het-NOD","minutes:  10<br />glucose:  NA<br />genotype: Het-NOD","minutes:  10<br />glucose:  NA<br />genotype: Het-NOD","minutes:  15<br />glucose: 161<br />genotype: Het-NOD","minutes:  15<br />glucose: 155<br />genotype: Het-NOD","minutes:  30<br />glucose: 157<br />genotype: Het-NOD","minutes:  30<br />glucose:  90<br />genotype: Het-NOD","minutes:  45<br />glucose:  NA<br />genotype: Het-NOD","minutes:  45<br />glucose:  NA<br />genotype: Het-NOD","minutes:  60<br />glucose: 131<br />genotype: Het-NOD","minutes:  60<br />glucose:  63<br />genotype: Het-NOD","minutes:  75<br />glucose:  NA<br />genotype: Het-NOD","minutes:  75<br />glucose:  NA<br />genotype: Het-NOD","minutes:  90<br />glucose: 114<br />genotype: Het-NOD","minutes:  90<br />glucose:  NA<br />genotype: Het-NOD","minutes: 120<br />glucose:  NA<br />genotype: Het-NOD","minutes: 120<br />glucose:  NA<br />genotype: Het-NOD"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(213,94,0,1)","opacity":0.5,"size":5.6692913385826778,"symbol":"circle","line":{"width":1.8897637795275593,"color":"rgba(213,94,0,1)"}},"hoveron":"points","name":"Het-NOD","legendgroup":"Het-NOD","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,1,1,3,3,3,3,3,3,3,3,3,3,5,5,5,5,5,5,5,5,5,5,10,10,10,10,10,10,10,10,10,10,15,15,15,15,15,15,15,15,15,15,30,30,30,30,30,30,30,30,30,30,45,45,45,45,45,45,45,45,45,45,60,60,60,60,60,60,60,60,60,60,75,75,75,75,75,75,75,75,75,75,90,90,90,90,90,90,90,90,90,90,120,120,120,120,120,120,120,120,120,120],"y":[114,124,153,115,123,198,129,122,112,134,null,123,136,111,124,137,120,106,106,121,null,null,null,127,112,152,121,106,119,137,null,98,124,130,111,150,111,120,125,108,null,101,142,118,96,147,85,83,104,123,164,61,76,121,83,99,62,61,77,101,141,57,104,71,63,61,45,95,67,81,null,39,null,null,null,null,null,null,null,null,109,null,48,96,33,64,32,67,56,104,null,45,null,null,null,null,null,null,null,null,null,null,30,118,32,44,38,66,44,117,null,45,35,81,33,40,37,89,55,126],"text":["minutes:   0<br />glucose: 114<br />genotype: KO-NOD","minutes:   0<br />glucose: 124<br />genotype: KO-NOD","minutes:   0<br />glucose: 153<br />genotype: KO-NOD","minutes:   0<br />glucose: 115<br />genotype: KO-NOD","minutes:   0<br />glucose: 123<br />genotype: KO-NOD","minutes:   0<br />glucose: 198<br />genotype: KO-NOD","minutes:   0<br />glucose: 129<br />genotype: KO-NOD","minutes:   0<br />glucose: 122<br />genotype: KO-NOD","minutes:   0<br />glucose: 112<br />genotype: KO-NOD","minutes:   0<br />glucose: 134<br />genotype: KO-NOD","minutes:   1<br />glucose:  NA<br />genotype: KO-NOD","minutes:   1<br />glucose: 123<br />genotype: KO-NOD","minutes:   1<br />glucose: 136<br />genotype: KO-NOD","minutes:   1<br />glucose: 111<br />genotype: KO-NOD","minutes:   1<br />glucose: 124<br />genotype: KO-NOD","minutes:   1<br />glucose: 137<br />genotype: KO-NOD","minutes:   1<br />glucose: 120<br />genotype: KO-NOD","minutes:   1<br />glucose: 106<br />genotype: KO-NOD","minutes:   1<br />glucose: 106<br />genotype: KO-NOD","minutes:   1<br />glucose: 121<br />genotype: KO-NOD","minutes:   3<br />glucose:  NA<br />genotype: KO-NOD","minutes:   3<br />glucose:  NA<br />genotype: KO-NOD","minutes:   3<br />glucose:  NA<br />genotype: KO-NOD","minutes:   3<br />glucose: 127<br />genotype: KO-NOD","minutes:   3<br />glucose: 112<br />genotype: KO-NOD","minutes:   3<br />glucose: 152<br />genotype: KO-NOD","minutes:   3<br />glucose: 121<br />genotype: KO-NOD","minutes:   3<br />glucose: 106<br />genotype: KO-NOD","minutes:   3<br />glucose: 119<br />genotype: KO-NOD","minutes:   3<br />glucose: 137<br />genotype: KO-NOD","minutes:   5<br />glucose:  NA<br />genotype: KO-NOD","minutes:   5<br />glucose:  98<br />genotype: KO-NOD","minutes:   5<br />glucose: 124<br />genotype: KO-NOD","minutes:   5<br />glucose: 130<br />genotype: KO-NOD","minutes:   5<br />glucose: 111<br />genotype: KO-NOD","minutes:   5<br />glucose: 150<br />genotype: KO-NOD","minutes:   5<br />glucose: 111<br />genotype: KO-NOD","minutes:   5<br />glucose: 120<br />genotype: KO-NOD","minutes:   5<br />glucose: 125<br />genotype: KO-NOD","minutes:   5<br />glucose: 108<br />genotype: KO-NOD","minutes:  10<br />glucose:  NA<br />genotype: KO-NOD","minutes:  10<br />glucose: 101<br />genotype: KO-NOD","minutes:  10<br />glucose: 142<br />genotype: KO-NOD","minutes:  10<br />glucose: 118<br />genotype: KO-NOD","minutes:  10<br />glucose:  96<br />genotype: KO-NOD","minutes:  10<br />glucose: 147<br />genotype: KO-NOD","minutes:  10<br />glucose:  85<br />genotype: KO-NOD","minutes:  10<br />glucose:  83<br />genotype: KO-NOD","minutes:  10<br />glucose: 104<br />genotype: KO-NOD","minutes:  10<br />glucose: 123<br />genotype: KO-NOD","minutes:  15<br />glucose: 164<br />genotype: KO-NOD","minutes:  15<br />glucose:  61<br />genotype: KO-NOD","minutes:  15<br />glucose:  76<br />genotype: KO-NOD","minutes:  15<br />glucose: 121<br />genotype: KO-NOD","minutes:  15<br />glucose:  83<br />genotype: KO-NOD","minutes:  15<br />glucose:  99<br />genotype: KO-NOD","minutes:  15<br />glucose:  62<br />genotype: KO-NOD","minutes:  15<br />glucose:  61<br />genotype: KO-NOD","minutes:  15<br />glucose:  77<br />genotype: KO-NOD","minutes:  15<br />glucose: 101<br />genotype: KO-NOD","minutes:  30<br />glucose: 141<br />genotype: KO-NOD","minutes:  30<br />glucose:  57<br />genotype: KO-NOD","minutes:  30<br />glucose: 104<br />genotype: KO-NOD","minutes:  30<br />glucose:  71<br />genotype: KO-NOD","minutes:  30<br />glucose:  63<br />genotype: KO-NOD","minutes:  30<br />glucose:  61<br />genotype: KO-NOD","minutes:  30<br />glucose:  45<br />genotype: KO-NOD","minutes:  30<br />glucose:  95<br />genotype: KO-NOD","minutes:  30<br />glucose:  67<br />genotype: KO-NOD","minutes:  30<br />glucose:  81<br />genotype: KO-NOD","minutes:  45<br />glucose:  NA<br />genotype: KO-NOD","minutes:  45<br />glucose:  39<br />genotype: KO-NOD","minutes:  45<br />glucose:  NA<br />genotype: KO-NOD","minutes:  45<br />glucose:  NA<br />genotype: KO-NOD","minutes:  45<br />glucose:  NA<br />genotype: KO-NOD","minutes:  45<br />glucose:  NA<br />genotype: KO-NOD","minutes:  45<br />glucose:  NA<br />genotype: KO-NOD","minutes:  45<br />glucose:  NA<br />genotype: KO-NOD","minutes:  45<br />glucose:  NA<br />genotype: KO-NOD","minutes:  45<br />glucose:  NA<br />genotype: KO-NOD","minutes:  60<br />glucose: 109<br />genotype: KO-NOD","minutes:  60<br />glucose:  NA<br />genotype: KO-NOD","minutes:  60<br />glucose:  48<br />genotype: KO-NOD","minutes:  60<br />glucose:  96<br />genotype: KO-NOD","minutes:  60<br />glucose:  33<br />genotype: KO-NOD","minutes:  60<br />glucose:  64<br />genotype: KO-NOD","minutes:  60<br />glucose:  32<br />genotype: KO-NOD","minutes:  60<br />glucose:  67<br />genotype: KO-NOD","minutes:  60<br />glucose:  56<br />genotype: KO-NOD","minutes:  60<br />glucose: 104<br />genotype: KO-NOD","minutes:  75<br />glucose:  NA<br />genotype: KO-NOD","minutes:  75<br />glucose:  45<br />genotype: KO-NOD","minutes:  75<br />glucose:  NA<br />genotype: KO-NOD","minutes:  75<br />glucose:  NA<br />genotype: KO-NOD","minutes:  75<br />glucose:  NA<br />genotype: KO-NOD","minutes:  75<br />glucose:  NA<br />genotype: KO-NOD","minutes:  75<br />glucose:  NA<br />genotype: KO-NOD","minutes:  75<br />glucose:  NA<br />genotype: KO-NOD","minutes:  75<br />glucose:  NA<br />genotype: KO-NOD","minutes:  75<br />glucose:  NA<br />genotype: KO-NOD","minutes:  90<br />glucose:  NA<br />genotype: KO-NOD","minutes:  90<br />glucose:  NA<br />genotype: KO-NOD","minutes:  90<br />glucose:  30<br />genotype: KO-NOD","minutes:  90<br />glucose: 118<br />genotype: KO-NOD","minutes:  90<br />glucose:  32<br />genotype: KO-NOD","minutes:  90<br />glucose:  44<br />genotype: KO-NOD","minutes:  90<br />glucose:  38<br />genotype: KO-NOD","minutes:  90<br />glucose:  66<br />genotype: KO-NOD","minutes:  90<br />glucose:  44<br />genotype: KO-NOD","minutes:  90<br />glucose: 117<br />genotype: KO-NOD","minutes: 120<br />glucose:  NA<br />genotype: KO-NOD","minutes: 120<br />glucose:  45<br />genotype: KO-NOD","minutes: 120<br />glucose:  35<br />genotype: KO-NOD","minutes: 120<br />glucose:  81<br />genotype: KO-NOD","minutes: 120<br />glucose:  33<br />genotype: KO-NOD","minutes: 120<br />glucose:  40<br />genotype: KO-NOD","minutes: 120<br />glucose:  37<br />genotype: KO-NOD","minutes: 120<br />glucose:  89<br />genotype: KO-NOD","minutes: 120<br />glucose:  55<br />genotype: KO-NOD","minutes: 120<br />glucose: 126<br />genotype: KO-NOD"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(86,180,233,1)","opacity":0.5,"size":5.6692913385826778,"symbol":"circle","line":{"width":1.8897637795275593,"color":"rgba(86,180,233,1)"}},"hoveron":"points","name":"KO-NOD","legendgroup":"KO-NOD","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1.518987341772152,3.037974683544304,4.556962025316456,6.075949367088608,7.59493670886076,9.113924050632912,10.632911392405063,12.151898734177216,13.670886075949369,15.18987341772152,16.708860759493671,18.227848101265824,19.746835443037977,21.265822784810126,22.784810126582279,24.303797468354432,25.822784810126585,27.341772151898738,28.860759493670887,30.37974683544304,31.898734177215193,33.417721518987342,34.936708860759495,36.455696202531648,37.974683544303801,39.493670886075954,41.012658227848107,42.531645569620252,44.050632911392405,45.569620253164558,47.088607594936711,48.607594936708864,50.126582278481017,51.64556962025317,53.164556962025323,54.683544303797476,56.202531645569621,57.721518987341774,59.240506329113927,60.75949367088608,62.278481012658233,63.797468354430386,65.316455696202539,66.835443037974684,68.354430379746844,69.87341772151899,71.39240506329115,72.911392405063296,74.430379746835442,75.949367088607602,77.468354430379748,78.987341772151908,80.506329113924053,82.025316455696213,83.544303797468359,85.063291139240505,86.582278481012665,88.101265822784811,89.620253164556971,91.139240506329116,92.658227848101276,94.177215189873422,95.696202531645582,97.215189873417728,98.734177215189874,100.25316455696203,101.77215189873418,103.29113924050634,104.81012658227849,106.32911392405065,107.84810126582279,109.36708860759495,110.8860759493671,112.40506329113924,113.9240506329114,115.44303797468355,116.96202531645571,118.48101265822785,120],"y":[207.12317025005558,206.66960380193663,206.21603735381768,205.76247090569873,205.30890445757979,204.85533800946084,204.40177156134186,203.94820511322291,203.49463866510396,203.04107221698501,202.58750576886607,202.13393932074712,201.68037287262817,201.22680642450922,200.77323997639027,200.31967352827132,199.86610708015235,199.4125406320334,198.95897418391445,198.5054077357955,198.05184128767655,197.5982748395576,197.14470839143866,196.69114194331971,196.23757549520076,195.78400904708181,195.33044259896286,194.87687615084388,194.42330970272494,193.96974325460599,193.51617680648704,193.06261035836809,192.60904391024914,192.15547746213019,191.70191101401124,191.2483445658923,190.79477811777335,190.3412116696544,189.88764522153542,189.43407877341647,188.98051232529752,188.52694587717858,188.07337942905963,187.61981298094068,187.16624653282173,186.71268008470278,186.25911363658383,185.80554718846486,185.35198074034594,184.89841429222696,184.44484784410801,183.99128139598906,183.53771494787011,183.08414849975117,182.63058205163222,182.17701560351327,181.72344915539432,181.26988270727537,180.81631625915642,180.36274981103747,179.9091833629185,179.45561691479955,179.0020504666806,178.54848401856165,178.0949175704427,177.64135112232375,177.18778467420481,176.73421822608586,176.28065177796691,175.82708532984796,175.37351888172898,174.91995243361004,174.46638598549109,174.01281953737214,173.55925308925319,173.10568664113424,172.65212019301529,172.19855374489634,171.7449872967774,171.29142084865845],"text":["minutes:   0.000000<br />glucose: 207.12317<br />genotype: WT-WT","minutes:   1.518987<br />glucose: 206.66960<br />genotype: WT-WT","minutes:   3.037975<br />glucose: 206.21604<br />genotype: WT-WT","minutes:   4.556962<br />glucose: 205.76247<br />genotype: WT-WT","minutes:   6.075949<br />glucose: 205.30890<br />genotype: WT-WT","minutes:   7.594937<br />glucose: 204.85534<br />genotype: WT-WT","minutes:   9.113924<br />glucose: 204.40177<br />genotype: WT-WT","minutes:  10.632911<br />glucose: 203.94821<br />genotype: WT-WT","minutes:  12.151899<br />glucose: 203.49464<br />genotype: WT-WT","minutes:  13.670886<br />glucose: 203.04107<br />genotype: WT-WT","minutes:  15.189873<br />glucose: 202.58751<br />genotype: WT-WT","minutes:  16.708861<br />glucose: 202.13394<br />genotype: WT-WT","minutes:  18.227848<br />glucose: 201.68037<br />genotype: WT-WT","minutes:  19.746835<br />glucose: 201.22681<br />genotype: WT-WT","minutes:  21.265823<br />glucose: 200.77324<br />genotype: WT-WT","minutes:  22.784810<br />glucose: 200.31967<br />genotype: WT-WT","minutes:  24.303797<br />glucose: 199.86611<br />genotype: WT-WT","minutes:  25.822785<br />glucose: 199.41254<br />genotype: WT-WT","minutes:  27.341772<br />glucose: 198.95897<br />genotype: WT-WT","minutes:  28.860759<br />glucose: 198.50541<br />genotype: WT-WT","minutes:  30.379747<br />glucose: 198.05184<br />genotype: WT-WT","minutes:  31.898734<br />glucose: 197.59827<br />genotype: WT-WT","minutes:  33.417722<br />glucose: 197.14471<br />genotype: WT-WT","minutes:  34.936709<br />glucose: 196.69114<br />genotype: WT-WT","minutes:  36.455696<br />glucose: 196.23758<br />genotype: WT-WT","minutes:  37.974684<br />glucose: 195.78401<br />genotype: WT-WT","minutes:  39.493671<br />glucose: 195.33044<br />genotype: WT-WT","minutes:  41.012658<br />glucose: 194.87688<br />genotype: WT-WT","minutes:  42.531646<br />glucose: 194.42331<br />genotype: WT-WT","minutes:  44.050633<br />glucose: 193.96974<br />genotype: WT-WT","minutes:  45.569620<br />glucose: 193.51618<br />genotype: WT-WT","minutes:  47.088608<br />glucose: 193.06261<br />genotype: WT-WT","minutes:  48.607595<br />glucose: 192.60904<br />genotype: WT-WT","minutes:  50.126582<br />glucose: 192.15548<br />genotype: WT-WT","minutes:  51.645570<br />glucose: 191.70191<br />genotype: WT-WT","minutes:  53.164557<br />glucose: 191.24834<br />genotype: WT-WT","minutes:  54.683544<br />glucose: 190.79478<br />genotype: WT-WT","minutes:  56.202532<br />glucose: 190.34121<br />genotype: WT-WT","minutes:  57.721519<br />glucose: 189.88765<br />genotype: WT-WT","minutes:  59.240506<br />glucose: 189.43408<br />genotype: WT-WT","minutes:  60.759494<br />glucose: 188.98051<br />genotype: WT-WT","minutes:  62.278481<br />glucose: 188.52695<br />genotype: WT-WT","minutes:  63.797468<br />glucose: 188.07338<br />genotype: WT-WT","minutes:  65.316456<br />glucose: 187.61981<br />genotype: WT-WT","minutes:  66.835443<br />glucose: 187.16625<br />genotype: WT-WT","minutes:  68.354430<br />glucose: 186.71268<br />genotype: WT-WT","minutes:  69.873418<br />glucose: 186.25911<br />genotype: WT-WT","minutes:  71.392405<br />glucose: 185.80555<br />genotype: WT-WT","minutes:  72.911392<br />glucose: 185.35198<br />genotype: WT-WT","minutes:  74.430380<br />glucose: 184.89841<br />genotype: WT-WT","minutes:  75.949367<br />glucose: 184.44485<br />genotype: WT-WT","minutes:  77.468354<br />glucose: 183.99128<br />genotype: WT-WT","minutes:  78.987342<br />glucose: 183.53771<br />genotype: WT-WT","minutes:  80.506329<br />glucose: 183.08415<br />genotype: WT-WT","minutes:  82.025316<br />glucose: 182.63058<br />genotype: WT-WT","minutes:  83.544304<br />glucose: 182.17702<br />genotype: WT-WT","minutes:  85.063291<br />glucose: 181.72345<br />genotype: WT-WT","minutes:  86.582278<br />glucose: 181.26988<br />genotype: WT-WT","minutes:  88.101266<br />glucose: 180.81632<br />genotype: WT-WT","minutes:  89.620253<br />glucose: 180.36275<br />genotype: WT-WT","minutes:  91.139241<br />glucose: 179.90918<br />genotype: WT-WT","minutes:  92.658228<br />glucose: 179.45562<br />genotype: WT-WT","minutes:  94.177215<br />glucose: 179.00205<br />genotype: WT-WT","minutes:  95.696203<br />glucose: 178.54848<br />genotype: WT-WT","minutes:  97.215190<br />glucose: 178.09492<br />genotype: WT-WT","minutes:  98.734177<br />glucose: 177.64135<br />genotype: WT-WT","minutes: 100.253165<br />glucose: 177.18778<br />genotype: WT-WT","minutes: 101.772152<br />glucose: 176.73422<br />genotype: WT-WT","minutes: 103.291139<br />glucose: 176.28065<br />genotype: WT-WT","minutes: 104.810127<br />glucose: 175.82709<br />genotype: WT-WT","minutes: 106.329114<br />glucose: 175.37352<br />genotype: WT-WT","minutes: 107.848101<br />glucose: 174.91995<br />genotype: WT-WT","minutes: 109.367089<br />glucose: 174.46639<br />genotype: WT-WT","minutes: 110.886076<br />glucose: 174.01282<br />genotype: WT-WT","minutes: 112.405063<br />glucose: 173.55925<br />genotype: WT-WT","minutes: 113.924051<br />glucose: 173.10569<br />genotype: WT-WT","minutes: 115.443038<br />glucose: 172.65212<br />genotype: WT-WT","minutes: 116.962025<br />glucose: 172.19855<br />genotype: WT-WT","minutes: 118.481013<br />glucose: 171.74499<br />genotype: WT-WT","minutes: 120.000000<br />glucose: 171.29142<br />genotype: WT-WT"],"type":"scatter","mode":"lines","name":"WT-WT","line":{"width":3.7795275590551185,"color":"rgba(0,158,115,1)","dash":"solid"},"hoveron":"points","legendgroup":"WT-WT","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1.139240506329114,2.278481012658228,3.4177215189873422,4.556962025316456,5.6962025316455698,6.8354430379746844,7.9746835443037982,9.113924050632912,10.253164556962027,11.39240506329114,12.531645569620254,13.670886075949369,14.810126582278482,15.949367088607596,17.088607594936711,18.227848101265824,19.367088607594937,20.506329113924053,21.645569620253166,22.784810126582279,23.924050632911396,25.063291139240508,26.202531645569621,27.341772151898738,28.481012658227851,29.620253164556964,30.759493670886076,31.898734177215193,33.037974683544306,34.177215189873422,35.316455696202532,36.455696202531648,37.594936708860764,38.734177215189874,39.87341772151899,41.012658227848107,42.151898734177216,43.291139240506332,44.430379746835449,45.569620253164558,46.708860759493675,47.848101265822791,48.9873417721519,50.126582278481017,51.265822784810133,52.405063291139243,53.544303797468359,54.683544303797476,55.822784810126585,56.962025316455701,58.101265822784811,59.240506329113927,60.379746835443044,61.518987341772153,62.658227848101269,63.797468354430386,64.936708860759495,66.075949367088612,67.215189873417728,68.354430379746844,69.493670886075961,70.632911392405063,71.77215189873418,72.911392405063296,74.050632911392412,75.189873417721529,76.329113924050631,77.468354430379748,78.607594936708864,79.74683544303798,80.886075949367097,82.025316455696213,83.164556962025316,84.303797468354432,85.443037974683548,86.582278481012665,87.721518987341781,88.860759493670898,90],"y":[133.25827814569536,132.87350155084249,132.4887249559896,132.10394836113673,131.71917176628384,131.33439517143097,130.94961857657808,130.56484198172521,130.18006538687231,129.79528879201945,129.41051219716655,129.02573560231369,128.64095900746079,128.25618241260793,127.87140581775505,127.48662922290217,127.10185262804929,126.71707603319641,126.33229943834353,125.94752284349065,125.56274624863777,125.17796965378488,124.793193058932,124.40841646407912,124.02363986922624,123.63886327437336,123.25408667952048,122.8693100846676,122.48453348981472,122.09975689496184,121.71498030010898,121.3302037052561,120.94542711040322,120.56065051555034,120.17587392069746,119.79109732584458,119.40632073099169,119.02154413613881,118.63676754128593,118.25199094643305,117.86721435158017,117.48243775672729,117.09766116187441,116.71288456702153,116.32810797216865,115.94333137731577,115.55855478246289,115.17377818761003,114.78900159275713,114.40422499790427,114.01944840305137,113.6346718081985,113.24989521334562,112.86511861849274,112.48034202363986,112.09556542878698,111.7107888339341,111.32601223908122,110.94123564422834,110.55645904937546,110.17168245452258,109.7869058596697,109.40212926481682,109.01735266996394,108.63257607511106,108.24779948025818,107.8630228854053,107.47824629055242,107.09346969569955,106.70869310084666,106.32391650599379,105.9391399111409,105.55436331628803,105.16958672143515,104.78481012658227,104.40003353172939,104.01525693687651,103.63048034202363,103.24570374717075,102.86092715231787],"text":["minutes:   0.000000<br />glucose: 133.25828<br />genotype: Het-NOD","minutes:   1.139241<br />glucose: 132.87350<br />genotype: Het-NOD","minutes:   2.278481<br />glucose: 132.48872<br />genotype: Het-NOD","minutes:   3.417722<br />glucose: 132.10395<br />genotype: Het-NOD","minutes:   4.556962<br />glucose: 131.71917<br />genotype: Het-NOD","minutes:   5.696203<br />glucose: 131.33440<br />genotype: Het-NOD","minutes:   6.835443<br />glucose: 130.94962<br />genotype: Het-NOD","minutes:   7.974684<br />glucose: 130.56484<br />genotype: Het-NOD","minutes:   9.113924<br />glucose: 130.18007<br />genotype: Het-NOD","minutes:  10.253165<br />glucose: 129.79529<br />genotype: Het-NOD","minutes:  11.392405<br />glucose: 129.41051<br />genotype: Het-NOD","minutes:  12.531646<br />glucose: 129.02574<br />genotype: Het-NOD","minutes:  13.670886<br />glucose: 128.64096<br />genotype: Het-NOD","minutes:  14.810127<br />glucose: 128.25618<br />genotype: Het-NOD","minutes:  15.949367<br />glucose: 127.87141<br />genotype: Het-NOD","minutes:  17.088608<br />glucose: 127.48663<br />genotype: Het-NOD","minutes:  18.227848<br />glucose: 127.10185<br />genotype: Het-NOD","minutes:  19.367089<br />glucose: 126.71708<br />genotype: Het-NOD","minutes:  20.506329<br />glucose: 126.33230<br />genotype: Het-NOD","minutes:  21.645570<br />glucose: 125.94752<br />genotype: Het-NOD","minutes:  22.784810<br />glucose: 125.56275<br />genotype: Het-NOD","minutes:  23.924051<br />glucose: 125.17797<br />genotype: Het-NOD","minutes:  25.063291<br />glucose: 124.79319<br />genotype: Het-NOD","minutes:  26.202532<br />glucose: 124.40842<br />genotype: Het-NOD","minutes:  27.341772<br />glucose: 124.02364<br />genotype: Het-NOD","minutes:  28.481013<br />glucose: 123.63886<br />genotype: Het-NOD","minutes:  29.620253<br />glucose: 123.25409<br />genotype: Het-NOD","minutes:  30.759494<br />glucose: 122.86931<br />genotype: Het-NOD","minutes:  31.898734<br />glucose: 122.48453<br />genotype: Het-NOD","minutes:  33.037975<br />glucose: 122.09976<br />genotype: Het-NOD","minutes:  34.177215<br />glucose: 121.71498<br />genotype: Het-NOD","minutes:  35.316456<br />glucose: 121.33020<br />genotype: Het-NOD","minutes:  36.455696<br />glucose: 120.94543<br />genotype: Het-NOD","minutes:  37.594937<br />glucose: 120.56065<br />genotype: Het-NOD","minutes:  38.734177<br />glucose: 120.17587<br />genotype: Het-NOD","minutes:  39.873418<br />glucose: 119.79110<br />genotype: Het-NOD","minutes:  41.012658<br />glucose: 119.40632<br />genotype: Het-NOD","minutes:  42.151899<br />glucose: 119.02154<br />genotype: Het-NOD","minutes:  43.291139<br />glucose: 118.63677<br />genotype: Het-NOD","minutes:  44.430380<br />glucose: 118.25199<br />genotype: Het-NOD","minutes:  45.569620<br />glucose: 117.86721<br />genotype: Het-NOD","minutes:  46.708861<br />glucose: 117.48244<br />genotype: Het-NOD","minutes:  47.848101<br />glucose: 117.09766<br />genotype: Het-NOD","minutes:  48.987342<br />glucose: 116.71288<br />genotype: Het-NOD","minutes:  50.126582<br />glucose: 116.32811<br />genotype: Het-NOD","minutes:  51.265823<br />glucose: 115.94333<br />genotype: Het-NOD","minutes:  52.405063<br />glucose: 115.55855<br />genotype: Het-NOD","minutes:  53.544304<br />glucose: 115.17378<br />genotype: Het-NOD","minutes:  54.683544<br />glucose: 114.78900<br />genotype: Het-NOD","minutes:  55.822785<br />glucose: 114.40422<br />genotype: Het-NOD","minutes:  56.962025<br />glucose: 114.01945<br />genotype: Het-NOD","minutes:  58.101266<br />glucose: 113.63467<br />genotype: Het-NOD","minutes:  59.240506<br />glucose: 113.24990<br />genotype: Het-NOD","minutes:  60.379747<br />glucose: 112.86512<br />genotype: Het-NOD","minutes:  61.518987<br />glucose: 112.48034<br />genotype: Het-NOD","minutes:  62.658228<br />glucose: 112.09557<br />genotype: Het-NOD","minutes:  63.797468<br />glucose: 111.71079<br />genotype: Het-NOD","minutes:  64.936709<br />glucose: 111.32601<br />genotype: Het-NOD","minutes:  66.075949<br />glucose: 110.94124<br />genotype: Het-NOD","minutes:  67.215190<br />glucose: 110.55646<br />genotype: Het-NOD","minutes:  68.354430<br />glucose: 110.17168<br />genotype: Het-NOD","minutes:  69.493671<br />glucose: 109.78691<br />genotype: Het-NOD","minutes:  70.632911<br />glucose: 109.40213<br />genotype: Het-NOD","minutes:  71.772152<br />glucose: 109.01735<br />genotype: Het-NOD","minutes:  72.911392<br />glucose: 108.63258<br />genotype: Het-NOD","minutes:  74.050633<br />glucose: 108.24780<br />genotype: Het-NOD","minutes:  75.189873<br />glucose: 107.86302<br />genotype: Het-NOD","minutes:  76.329114<br />glucose: 107.47825<br />genotype: Het-NOD","minutes:  77.468354<br />glucose: 107.09347<br />genotype: Het-NOD","minutes:  78.607595<br />glucose: 106.70869<br />genotype: Het-NOD","minutes:  79.746835<br />glucose: 106.32392<br />genotype: Het-NOD","minutes:  80.886076<br />glucose: 105.93914<br />genotype: Het-NOD","minutes:  82.025316<br />glucose: 105.55436<br />genotype: Het-NOD","minutes:  83.164557<br />glucose: 105.16959<br />genotype: Het-NOD","minutes:  84.303797<br />glucose: 104.78481<br />genotype: Het-NOD","minutes:  85.443038<br />glucose: 104.40003<br />genotype: Het-NOD","minutes:  86.582278<br />glucose: 104.01526<br />genotype: Het-NOD","minutes:  87.721519<br />glucose: 103.63048<br />genotype: Het-NOD","minutes:  88.860759<br />glucose: 103.24570<br />genotype: Het-NOD","minutes:  90.000000<br />glucose: 102.86093<br />genotype: Het-NOD"],"type":"scatter","mode":"lines","name":"Het-NOD","line":{"width":3.7795275590551185,"color":"rgba(213,94,0,1)","dash":"solid"},"hoveron":"points","legendgroup":"Het-NOD","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1.518987341772152,3.037974683544304,4.556962025316456,6.075949367088608,7.59493670886076,9.113924050632912,10.632911392405063,12.151898734177216,13.670886075949369,15.18987341772152,16.708860759493671,18.227848101265824,19.746835443037977,21.265822784810126,22.784810126582279,24.303797468354432,25.822784810126585,27.341772151898738,28.860759493670887,30.37974683544304,31.898734177215193,33.417721518987342,34.936708860759495,36.455696202531648,37.974683544303801,39.493670886075954,41.012658227848107,42.531645569620252,44.050632911392405,45.569620253164558,47.088607594936711,48.607594936708864,50.126582278481017,51.64556962025317,53.164556962025323,54.683544303797476,56.202531645569621,57.721518987341774,59.240506329113927,60.75949367088608,62.278481012658233,63.797468354430386,65.316455696202539,66.835443037974684,68.354430379746844,69.87341772151899,71.39240506329115,72.911392405063296,74.430379746835442,75.949367088607602,77.468354430379748,78.987341772151908,80.506329113924053,82.025316455696213,83.544303797468359,85.063291139240505,86.582278481012665,88.101265822784811,89.620253164556971,91.139240506329116,92.658227848101276,94.177215189873422,95.696202531645582,97.215189873417728,98.734177215189874,100.25316455696203,101.77215189873418,103.29113924050634,104.81012658227849,106.32911392405065,107.84810126582279,109.36708860759495,110.8860759493671,112.40506329113924,113.9240506329114,115.44303797468355,116.96202531645571,118.48101265822785,120],"y":[115.36441577841421,114.45424535464683,113.54407493087943,112.63390450711205,111.72373408334467,110.81356365957728,109.90339323580989,108.9932228120425,108.08305238827512,107.17288196450772,106.26271154074034,105.35254111697296,104.44237069320558,103.53220026943818,102.6220298456708,101.71185942190341,100.80168899813603,99.891518574368632,98.981348150601249,98.071177726833866,97.161007303066469,96.250836879299086,95.340666455531704,94.430496031764307,93.520325607996924,92.610155184229541,91.699984760462144,90.789814336694761,89.879643912927378,88.969473489159995,88.059303065392612,87.149132641625215,86.238962217857832,85.328791794090449,84.418621370323052,83.508450946555669,82.598280522788286,81.688110099020889,80.777939675253521,79.867769251486123,78.957598827718726,78.047428403951358,77.137257980183961,76.227087556416564,75.316917132649195,74.406746708881798,73.496576285114415,72.586405861347032,71.676235437579635,70.766065013812266,69.855894590044869,68.945724166277472,68.035553742510089,67.125383318742706,66.215212894975309,65.30504247120794,64.394872047440543,63.48470162367316,62.574531199905778,61.664360776138381,60.754190352370998,59.844019928603608,58.933849504836225,58.023679081068835,57.113508657301452,56.203338233534069,55.293167809766672,54.382997385999289,53.472826962231899,52.562656538464516,51.652486114697126,50.742315690929743,49.832145267162346,48.921974843394963,48.01180441962758,47.101633995860198,46.191463572092815,45.281293148325418,44.371122724558035,43.460952300790652],"text":["minutes:   0.000000<br />glucose: 115.36442<br />genotype: KO-NOD","minutes:   1.518987<br />glucose: 114.45425<br />genotype: KO-NOD","minutes:   3.037975<br />glucose: 113.54407<br />genotype: KO-NOD","minutes:   4.556962<br />glucose: 112.63390<br />genotype: KO-NOD","minutes:   6.075949<br />glucose: 111.72373<br />genotype: KO-NOD","minutes:   7.594937<br />glucose: 110.81356<br />genotype: KO-NOD","minutes:   9.113924<br />glucose: 109.90339<br />genotype: KO-NOD","minutes:  10.632911<br />glucose: 108.99322<br />genotype: KO-NOD","minutes:  12.151899<br />glucose: 108.08305<br />genotype: KO-NOD","minutes:  13.670886<br />glucose: 107.17288<br />genotype: KO-NOD","minutes:  15.189873<br />glucose: 106.26271<br />genotype: KO-NOD","minutes:  16.708861<br />glucose: 105.35254<br />genotype: KO-NOD","minutes:  18.227848<br />glucose: 104.44237<br />genotype: KO-NOD","minutes:  19.746835<br />glucose: 103.53220<br />genotype: KO-NOD","minutes:  21.265823<br />glucose: 102.62203<br />genotype: KO-NOD","minutes:  22.784810<br />glucose: 101.71186<br />genotype: KO-NOD","minutes:  24.303797<br />glucose: 100.80169<br />genotype: KO-NOD","minutes:  25.822785<br />glucose:  99.89152<br />genotype: KO-NOD","minutes:  27.341772<br />glucose:  98.98135<br />genotype: KO-NOD","minutes:  28.860759<br />glucose:  98.07118<br />genotype: KO-NOD","minutes:  30.379747<br />glucose:  97.16101<br />genotype: KO-NOD","minutes:  31.898734<br />glucose:  96.25084<br />genotype: KO-NOD","minutes:  33.417722<br />glucose:  95.34067<br />genotype: KO-NOD","minutes:  34.936709<br />glucose:  94.43050<br />genotype: KO-NOD","minutes:  36.455696<br />glucose:  93.52033<br />genotype: KO-NOD","minutes:  37.974684<br />glucose:  92.61016<br />genotype: KO-NOD","minutes:  39.493671<br />glucose:  91.69998<br />genotype: KO-NOD","minutes:  41.012658<br />glucose:  90.78981<br />genotype: KO-NOD","minutes:  42.531646<br />glucose:  89.87964<br />genotype: KO-NOD","minutes:  44.050633<br />glucose:  88.96947<br />genotype: KO-NOD","minutes:  45.569620<br />glucose:  88.05930<br />genotype: KO-NOD","minutes:  47.088608<br />glucose:  87.14913<br />genotype: KO-NOD","minutes:  48.607595<br />glucose:  86.23896<br />genotype: KO-NOD","minutes:  50.126582<br />glucose:  85.32879<br />genotype: KO-NOD","minutes:  51.645570<br />glucose:  84.41862<br />genotype: KO-NOD","minutes:  53.164557<br />glucose:  83.50845<br />genotype: KO-NOD","minutes:  54.683544<br />glucose:  82.59828<br />genotype: KO-NOD","minutes:  56.202532<br />glucose:  81.68811<br />genotype: KO-NOD","minutes:  57.721519<br />glucose:  80.77794<br />genotype: KO-NOD","minutes:  59.240506<br />glucose:  79.86777<br />genotype: KO-NOD","minutes:  60.759494<br />glucose:  78.95760<br />genotype: KO-NOD","minutes:  62.278481<br />glucose:  78.04743<br />genotype: KO-NOD","minutes:  63.797468<br />glucose:  77.13726<br />genotype: KO-NOD","minutes:  65.316456<br />glucose:  76.22709<br />genotype: KO-NOD","minutes:  66.835443<br />glucose:  75.31692<br />genotype: KO-NOD","minutes:  68.354430<br />glucose:  74.40675<br />genotype: KO-NOD","minutes:  69.873418<br />glucose:  73.49658<br />genotype: KO-NOD","minutes:  71.392405<br />glucose:  72.58641<br />genotype: KO-NOD","minutes:  72.911392<br />glucose:  71.67624<br />genotype: KO-NOD","minutes:  74.430380<br />glucose:  70.76607<br />genotype: KO-NOD","minutes:  75.949367<br />glucose:  69.85589<br />genotype: KO-NOD","minutes:  77.468354<br />glucose:  68.94572<br />genotype: KO-NOD","minutes:  78.987342<br />glucose:  68.03555<br />genotype: KO-NOD","minutes:  80.506329<br />glucose:  67.12538<br />genotype: KO-NOD","minutes:  82.025316<br />glucose:  66.21521<br />genotype: KO-NOD","minutes:  83.544304<br />glucose:  65.30504<br />genotype: KO-NOD","minutes:  85.063291<br />glucose:  64.39487<br />genotype: KO-NOD","minutes:  86.582278<br />glucose:  63.48470<br />genotype: KO-NOD","minutes:  88.101266<br />glucose:  62.57453<br />genotype: KO-NOD","minutes:  89.620253<br />glucose:  61.66436<br />genotype: KO-NOD","minutes:  91.139241<br />glucose:  60.75419<br />genotype: KO-NOD","minutes:  92.658228<br />glucose:  59.84402<br />genotype: KO-NOD","minutes:  94.177215<br />glucose:  58.93385<br />genotype: KO-NOD","minutes:  95.696203<br />glucose:  58.02368<br />genotype: KO-NOD","minutes:  97.215190<br />glucose:  57.11351<br />genotype: KO-NOD","minutes:  98.734177<br />glucose:  56.20334<br />genotype: KO-NOD","minutes: 100.253165<br />glucose:  55.29317<br />genotype: KO-NOD","minutes: 101.772152<br />glucose:  54.38300<br />genotype: KO-NOD","minutes: 103.291139<br />glucose:  53.47283<br />genotype: KO-NOD","minutes: 104.810127<br />glucose:  52.56266<br />genotype: KO-NOD","minutes: 106.329114<br />glucose:  51.65249<br />genotype: KO-NOD","minutes: 107.848101<br />glucose:  50.74232<br />genotype: KO-NOD","minutes: 109.367089<br />glucose:  49.83215<br />genotype: KO-NOD","minutes: 110.886076<br />glucose:  48.92197<br />genotype: KO-NOD","minutes: 112.405063<br />glucose:  48.01180<br />genotype: KO-NOD","minutes: 113.924051<br />glucose:  47.10163<br />genotype: KO-NOD","minutes: 115.443038<br />glucose:  46.19146<br />genotype: KO-NOD","minutes: 116.962025<br />glucose:  45.28129<br />genotype: KO-NOD","minutes: 118.481013<br />glucose:  44.37112<br />genotype: KO-NOD","minutes: 120.000000<br />glucose:  43.46095<br />genotype: KO-NOD"],"type":"scatter","mode":"lines","name":"KO-NOD","line":{"width":3.7795275590551185,"color":"rgba(86,180,233,1)","dash":"solid"},"hoveron":"points","legendgroup":"KO-NOD","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.762557077625573,"r":7.3059360730593621,"b":40.182648401826491,"l":43.105022831050235},"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"title":{"text":"<span style='color: #009E73 '>WT-WT<\/span> vs  <span style='color: #D55E00 '>Het-NOD<\/span> vs  <span style='color: #56B4E9 '>KO-NOD<\/span> Blood Glucose Levels in the First Week","font":{"color":"rgba(0,0,0,1)","family":"","size":17.534246575342465},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-6,126],"tickmode":"array","ticktext":["0","30","60","90","120"],"tickvals":[0,30,60,90,120],"categoryorder":"array","categoryarray":["0","30","60","90","120"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"y","title":{"text":"Test Minutes","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[18.399999999999999,273.60000000000002],"tickmode":"array","ticktext":["100","200"],"tickvals":[100,200],"categoryorder":"array","categoryarray":["100","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"x","title":{"text":"Blood Glucose","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5ee44665468a":{"x":{},"y":{},"colour":{},"type":"scatter"},"5ee41f2470e6":{"x":{},"y":{},"colour":{}}},"cur_data":"5ee44665468a","visdat":{"5ee44665468a":["function (y) ","x"],"5ee41f2470e6":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<section id="mixed-model" class="level2">
<h2 class="anchored" data-anchor-id="mixed-model">mixed model</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb41"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readr)</span>
<span id="cb41-2"><a href="#cb41-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span>
<span id="cb41-3"><a href="#cb41-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(lme4)</span>
<span id="cb41-4"><a href="#cb41-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb41-5"><a href="#cb41-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggeffects)</span>
<span id="cb41-6"><a href="#cb41-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-7"><a href="#cb41-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-8"><a href="#cb41-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-9"><a href="#cb41-9" aria-hidden="true" tabindex="-1"></a><span class="co"># change variables to factors and re-level variables</span></span>
<span id="cb41-10"><a href="#cb41-10" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="ot">&lt;-</span> clean2_IPGTT_data <span class="sc">%&gt;%</span> </span>
<span id="cb41-11"><a href="#cb41-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb41-12"><a href="#cb41-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">test_week_factor =</span> <span class="fu">factor</span>(test_week, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"1"</span>, <span class="st">"2"</span>, <span class="st">"3"</span>, <span class="st">"4"</span>, <span class="st">"5"</span>, <span class="st">"6"</span>, <span class="st">"7"</span>, <span class="st">"8"</span>, <span class="st">"9"</span>, <span class="st">"10"</span>, <span class="st">"11"</span>, <span class="st">"12"</span>)),</span>
<span id="cb41-13"><a href="#cb41-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">minutes_factor =</span> <span class="fu">factor</span>(minutes, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"1"</span>, <span class="st">"3"</span>, <span class="st">"5"</span>, <span class="st">"10"</span>, <span class="st">"15"</span>, <span class="st">"30"</span>, <span class="st">"60"</span>, <span class="st">"90"</span>, <span class="st">"120"</span>))</span>
<span id="cb41-14"><a href="#cb41-14" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb41-15"><a href="#cb41-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-16"><a href="#cb41-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-17"><a href="#cb41-17" aria-hidden="true" tabindex="-1"></a><span class="co"># filter just week 1</span></span>
<span id="cb41-18"><a href="#cb41-18" aria-hidden="true" tabindex="-1"></a>clean2_IPGTT_data <span class="sc">%&gt;%</span></span>
<span id="cb41-19"><a href="#cb41-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(test_week <span class="sc">==</span> <span class="dv">1</span>) <span class="sc">%&gt;%</span> <span class="fu">na.omit</span>() <span class="ot">-&gt;</span> week1_data_omit</span>
<span id="cb41-20"><a href="#cb41-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-21"><a href="#cb41-21" aria-hidden="true" tabindex="-1"></a><span class="co"># mixed-effects model1</span></span>
<span id="cb41-22"><a href="#cb41-22" aria-hidden="true" tabindex="-1"></a>mixed_model1 <span class="ot">&lt;-</span> <span class="fu">lmer</span>(glucose <span class="sc">~</span> minutes <span class="sc">*</span> genotype <span class="sc">+</span> </span>
<span id="cb41-23"><a href="#cb41-23" aria-hidden="true" tabindex="-1"></a>                      (<span class="dv">1</span><span class="sc">|</span>mouse_id),</span>
<span id="cb41-24"><a href="#cb41-24" aria-hidden="true" tabindex="-1"></a>                    <span class="at">data =</span> week1_data_omit)</span>
<span id="cb41-25"><a href="#cb41-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-26"><a href="#cb41-26" aria-hidden="true" tabindex="-1"></a><span class="co"># mixed-effects model2</span></span>
<span id="cb41-27"><a href="#cb41-27" aria-hidden="true" tabindex="-1"></a>mixed_model2 <span class="ot">&lt;-</span> <span class="fu">lmer</span>(glucose <span class="sc">~</span> minutes <span class="sc">+</span> test_week <span class="sc">+</span> genotype <span class="sc">+</span> </span>
<span id="cb41-28"><a href="#cb41-28" aria-hidden="true" tabindex="-1"></a>                      (<span class="dv">1</span><span class="sc">|</span>mouse_id) <span class="sc">+</span> </span>
<span id="cb41-29"><a href="#cb41-29" aria-hidden="true" tabindex="-1"></a>                      (<span class="dv">0</span> <span class="sc">+</span> test_week<span class="sc">|</span>mouse_id),</span>
<span id="cb41-30"><a href="#cb41-30" aria-hidden="true" tabindex="-1"></a>                    <span class="at">data =</span> clean2_IPGTT_data)</span>
<span id="cb41-31"><a href="#cb41-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-32"><a href="#cb41-32" aria-hidden="true" tabindex="-1"></a><span class="co"># summary</span></span>
<span id="cb41-33"><a href="#cb41-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-34"><a href="#cb41-34" aria-hidden="true" tabindex="-1"></a>emmm <span class="ot">&lt;-</span> <span class="fu">emmeans</span>(mixed_model1, <span class="at">specs =</span> <span class="sc">~</span> genotype <span class="sc">|</span> minutes)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Cannot use mode = "kenward-roger" because *pbkrtest* package is not installed</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Cannot use mode = "satterthwaite" because *lmerTest* package is not installed</code></pre>
</div>
<div class="sourceCode cell-code" id="cb44"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb44-1"><a href="#cb44-1" aria-hidden="true" tabindex="-1"></a>broom.mixed<span class="sc">::</span><span class="fu">tidy</span>(emmm)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 3 × 7
  genotype minutes estimate std.error    df statistic  p.value
  &lt;chr&gt;      &lt;dbl&gt;    &lt;dbl&gt;     &lt;dbl&gt; &lt;dbl&gt;     &lt;dbl&gt;    &lt;dbl&gt;
1 WT-WT       37.4    200.      12.9    Inf     15.4  8.10e-54
2 Het-NOD     37.4    127.      15.0    Inf      8.46 2.68e-17
3 KO-NOD      37.4     93.2      5.91   Inf     15.8  5.71e-56</code></pre>
</div>
<div class="sourceCode cell-code" id="cb46"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Predictions</span></span>
<span id="cb46-2"><a href="#cb46-2" aria-hidden="true" tabindex="-1"></a><span class="co"># predictions &lt;- ggpredict(mixed_model1, terms = c("minutes", "genotype", "test_week"))</span></span>
<span id="cb46-3"><a href="#cb46-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-4"><a href="#cb46-4" aria-hidden="true" tabindex="-1"></a><span class="co"># predictions as a data frame</span></span>
<span id="cb46-5"><a href="#cb46-5" aria-hidden="true" tabindex="-1"></a><span class="co"># pred_df &lt;- as.data.frame(predictions)</span></span>
<span id="cb46-6"><a href="#cb46-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-7"><a href="#cb46-7" aria-hidden="true" tabindex="-1"></a><span class="co"># differences</span></span>
<span id="cb46-8"><a href="#cb46-8" aria-hidden="true" tabindex="-1"></a>contrast_results <span class="ot">&lt;-</span> <span class="fu">pairs</span>(emmm, <span class="at">interaction =</span> <span class="st">"pairwise"</span>)</span>
<span id="cb46-9"><a href="#cb46-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-10"><a href="#cb46-10" aria-hidden="true" tabindex="-1"></a><span class="co"># convert to df</span></span>
<span id="cb46-11"><a href="#cb46-11" aria-hidden="true" tabindex="-1"></a>df_contrast <span class="ot">&lt;-</span> broom.mixed<span class="sc">::</span><span class="fu">tidy</span>(contrast_results)</span>
<span id="cb46-12"><a href="#cb46-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-13"><a href="#cb46-13" aria-hidden="true" tabindex="-1"></a>c <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(df_contrast, <span class="fu">aes</span>(<span class="at">x =</span> minutes, <span class="at">y =</span> estimate, <span class="at">color =</span> genotype_pairwise, <span class="at">group =</span> genotype_pairwise)) <span class="sc">+</span></span>
<span id="cb46-14"><a href="#cb46-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb46-15"><a href="#cb46-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb46-16"><a href="#cb46-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_errorbar</span>(<span class="fu">aes</span>(<span class="at">ymin =</span> estimate <span class="sc">-</span> std.error, <span class="at">ymax =</span> estimate <span class="sc">+</span> std.error), <span class="at">width =</span> <span class="fl">0.1</span>) <span class="sc">+</span></span>
<span id="cb46-17"><a href="#cb46-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb46-18"><a href="#cb46-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span> genotype_pairwise) <span class="sc">+</span></span>
<span id="cb46-19"><a href="#cb46-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"Differences in Means Between Genotypes (Week 1)"</span>,</span>
<span id="cb46-20"><a href="#cb46-20" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Time Point"</span>,</span>
<span id="cb46-21"><a href="#cb46-21" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Pairwise difference"</span>)</span>
<span id="cb46-22"><a href="#cb46-22" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(c)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="plotly html-widget html-fill-item" id="htmlwidget-274fb0d621bf34de8171" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-274fb0d621bf34de8171">{"x":{"data":[{"x":[37.361904761904761],"y":[33.774387491795004],"text":"minutes: 37.3619<br />estimate:  33.77439<br />genotype_pairwise: (Het-NOD) - (KO-NOD)<br />genotype_pairwise: (Het-NOD) - (KO-NOD)","type":"scatter","mode":"lines+markers","line":{"width":1.8897637795275593,"color":"rgba(248,118,109,1)","dash":"solid"},"hoveron":"points","name":"(Het-NOD) - (KO-NOD)","legendgroup":"(Het-NOD) - (KO-NOD)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","opacity":1,"size":5.6692913385826778,"symbol":"circle","line":{"width":1.8897637795275593,"color":"rgba(248,118,109,1)"}},"frame":null},{"x":[37.361904761904761],"y":[73.056541231767767],"text":"minutes: 37.3619<br />estimate:  73.05654<br />genotype_pairwise: (WT-WT) - (Het-NOD)<br />genotype_pairwise: (WT-WT) - (Het-NOD)","type":"scatter","mode":"lines+markers","line":{"width":1.8897637795275593,"color":"rgba(0,186,56,1)","dash":"solid"},"hoveron":"points","name":"(WT-WT) - (Het-NOD)","legendgroup":"(WT-WT) - (Het-NOD)","showlegend":true,"xaxis":"x2","yaxis":"y","hoverinfo":"text","marker":{"autocolorscale":false,"color":"rgba(0,186,56,1)","opacity":1,"size":5.6692913385826778,"symbol":"circle","line":{"width":1.8897637795275593,"color":"rgba(0,186,56,1)"}},"frame":null},{"x":[37.361904761904761],"y":[106.83092872356276],"text":"minutes: 37.3619<br />estimate: 106.83093<br />genotype_pairwise: (WT-WT) - (KO-NOD)<br />genotype_pairwise: (WT-WT) - (KO-NOD)","type":"scatter","mode":"lines+markers","line":{"width":1.8897637795275593,"color":"rgba(97,156,255,1)","dash":"solid"},"hoveron":"points","name":"(WT-WT) - (KO-NOD)","legendgroup":"(WT-WT) - (KO-NOD)","showlegend":true,"xaxis":"x3","yaxis":"y","hoverinfo":"text","marker":{"autocolorscale":false,"color":"rgba(97,156,255,1)","opacity":1,"size":5.6692913385826778,"symbol":"circle","line":{"width":1.8897637795275593,"color":"rgba(97,156,255,1)"}},"frame":null},{"x":[37.361904761904761],"y":[33.774387491795004],"text":"minutes: 37.3619<br />estimate:  33.77439<br />genotype_pairwise: (Het-NOD) - (KO-NOD)<br />genotype_pairwise: (Het-NOD) - (KO-NOD)<br />estimate - std.error: 17.64480<br />estimate + std.error:  49.90398","type":"scatter","mode":"lines","opacity":1,"line":{"color":"transparent"},"error_y":{"array":[16.129591191428176],"arrayminus":[16.129591191428176],"type":"data","width":407.27272727270622,"symmetric":false,"color":"rgba(248,118,109,1)"},"name":"(Het-NOD) - (KO-NOD)","legendgroup":"(Het-NOD) - (KO-NOD)","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[37.361904761904761],"y":[73.056541231767767],"text":"minutes: 37.3619<br />estimate:  73.05654<br />genotype_pairwise: (WT-WT) - (Het-NOD)<br />genotype_pairwise: (WT-WT) - (Het-NOD)<br />estimate - std.error: 53.23469<br />estimate + std.error:  92.87840","type":"scatter","mode":"lines","opacity":1,"line":{"color":"transparent"},"error_y":{"array":[19.821854270158909],"arrayminus":[19.821854270158909],"type":"data","width":407.27272727270622,"symmetric":false,"color":"rgba(0,186,56,1)"},"name":"(WT-WT) - (Het-NOD)","legendgroup":"(WT-WT) - (Het-NOD)","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[37.361904761904761],"y":[106.83092872356276],"text":"minutes: 37.3619<br />estimate: 106.83093<br />genotype_pairwise: (WT-WT) - (KO-NOD)<br />genotype_pairwise: (WT-WT) - (KO-NOD)<br />estimate - std.error: 92.59558<br />estimate + std.error: 121.06628","type":"scatter","mode":"lines","opacity":1,"line":{"color":"transparent"},"error_y":{"array":[14.235352488893852],"arrayminus":[14.235352488893852],"type":"data","width":407.27272727270622,"symmetric":false,"color":"rgba(97,156,255,1)"},"name":"(WT-WT) - (KO-NOD)","legendgroup":"(WT-WT) - (KO-NOD)","showlegend":false,"xaxis":"x3","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":55.452054794520549,"r":7.3059360730593621,"b":40.182648401826491,"l":43.105022831050235},"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"title":{"text":"Differences in Means Between Genotypes (Week 1)","font":{"color":"rgba(0,0,0,1)","family":"","size":17.534246575342465},"x":0,"xref":"paper"},"xaxis":{"domain":[0,0.32925636007827785],"automargin":true,"type":"linear","autorange":false,"range":[37.306904761904761,37.41690476190476],"tickmode":"array","ticktext":["37.325","37.350","37.375","37.400"],"tickvals":[37.325000000000003,37.350000000000001,37.375000000000007,37.400000000000006],"categoryorder":"array","categoryarray":["37.325","37.350","37.375","37.400"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"annotations":[{"text":"Time Point","x":0.5,"y":0,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"top","annotationType":"axis","yshift":-21.917808219178088},{"text":"Pairwise difference","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-27.762557077625573},{"text":"(Het-NOD) - (KO-NOD)","x":0.16462818003913893,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"(WT-WT) - (Het-NOD)","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"(WT-WT) - (KO-NOD)","x":0.83537181996086107,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[12.473722054762337,126.2373554580611],"tickmode":"array","ticktext":["25","50","75","100","125"],"tickvals":[25,50,75,100,125],"categoryorder":"array","categoryarray":["25","50","75","100","125"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":0.32925636007827785,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":0.32925636007827785,"y0":0,"y1":23.37899543378996,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0.33741030658838878,"x1":0.66258969341161122,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0.33741030658838878,"x1":0.66258969341161122,"y0":0,"y1":23.37899543378996,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0.67074363992172203,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0.67074363992172203,"x1":1,"y0":0,"y1":23.37899543378996,"yanchor":1,"ysizemode":"pixel"}],"xaxis2":{"type":"linear","autorange":false,"range":[37.306904761904761,37.41690476190476],"tickmode":"array","ticktext":["37.325","37.350","37.375","37.400"],"tickvals":[37.325000000000003,37.350000000000001,37.375000000000007,37.400000000000006],"categoryorder":"array","categoryarray":["37.325","37.350","37.375","37.400"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.33741030658838878,0.66258969341161122],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"xaxis3":{"type":"linear","autorange":false,"range":[37.306904761904761,37.41690476190476],"tickmode":"array","ticktext":["37.325","37.350","37.375","37.400"],"tickvals":[37.325000000000003,37.350000000000001,37.375000000000007,37.400000000000006],"categoryorder":"array","categoryarray":["37.325","37.350","37.375","37.400"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.67074363992172203,1],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"showlegend":true,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"title":{"text":"genotype_pairwise","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5ee48245318":{"x":{},"y":{},"colour":{},"type":"scatter"},"5ee4177173f8":{"x":{},"y":{},"colour":{}},"5ee4561c2b35":{"x":{},"y":{},"colour":{},"ymin":{},"ymax":{}}},"cur_data":"5ee48245318","visdat":{"5ee48245318":["function (y) ","x"],"5ee4177173f8":["function (y) ","x"],"5ee4561c2b35":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="new-data" class="level1">
<h1>new data</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb47"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb47-1"><a href="#cb47-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Load libraries</span></span>
<span id="cb47-2"><a href="#cb47-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb47-3"><a href="#cb47-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readxl)</span>
<span id="cb47-4"><a href="#cb47-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span>
<span id="cb47-5"><a href="#cb47-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(stringr)</span>
<span id="cb47-6"><a href="#cb47-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-7"><a href="#cb47-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Define path</span></span>
<span id="cb47-8"><a href="#cb47-8" aria-hidden="true" tabindex="-1"></a>glucose_tracking <span class="ot">&lt;-</span> <span class="fu">here</span>(<span class="st">"C:/Users/khopkins/Documents/mouseData/Glucose Tracking Results.xlsx"</span>)</span>
<span id="cb47-9"><a href="#cb47-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-10"><a href="#cb47-10" aria-hidden="true" tabindex="-1"></a><span class="co"># List sheet names</span></span>
<span id="cb47-11"><a href="#cb47-11" aria-hidden="true" tabindex="-1"></a>sheet_names <span class="ot">&lt;-</span> <span class="fu">excel_sheets</span>(glucose_tracking)</span>
<span id="cb47-12"><a href="#cb47-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-13"><a href="#cb47-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Filter sheets with "Group ##"</span></span>
<span id="cb47-14"><a href="#cb47-14" aria-hidden="true" tabindex="-1"></a>group_sheets <span class="ot">&lt;-</span> sheet_names[<span class="fu">str_detect</span>(sheet_names, <span class="st">"^Group</span><span class="sc">\\</span><span class="st">s?</span><span class="sc">\\</span><span class="st">d+$"</span>)]</span>
<span id="cb47-15"><a href="#cb47-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-16"><a href="#cb47-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Function for consistent date format</span></span>
<span id="cb47-17"><a href="#cb47-17" aria-hidden="true" tabindex="-1"></a>convert_to_date <span class="ot">&lt;-</span> <span class="cf">function</span>(data) {</span>
<span id="cb47-18"><a href="#cb47-18" aria-hidden="true" tabindex="-1"></a>  ymd_pattern <span class="ot">&lt;-</span> <span class="st">"^(</span><span class="sc">\\</span><span class="st">d{4})-(</span><span class="sc">\\</span><span class="st">d{2})-(</span><span class="sc">\\</span><span class="st">d{2})$"</span></span>
<span id="cb47-19"><a href="#cb47-19" aria-hidden="true" tabindex="-1"></a>  mdy_pattern <span class="ot">&lt;-</span> <span class="st">"^(</span><span class="sc">\\</span><span class="st">d{2})/(</span><span class="sc">\\</span><span class="st">d{2})/(</span><span class="sc">\\</span><span class="st">d{4})$"</span></span>
<span id="cb47-20"><a href="#cb47-20" aria-hidden="true" tabindex="-1"></a>  dmy_pattern <span class="ot">&lt;-</span> <span class="st">"^(</span><span class="sc">\\</span><span class="st">d{2})</span><span class="sc">\\</span><span class="st">.(</span><span class="sc">\\</span><span class="st">d{2})</span><span class="sc">\\</span><span class="st">.(</span><span class="sc">\\</span><span class="st">d{4})$"</span></span>
<span id="cb47-21"><a href="#cb47-21" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb47-22"><a href="#cb47-22" aria-hidden="true" tabindex="-1"></a>  data <span class="sc">%&gt;%</span></span>
<span id="cb47-23"><a href="#cb47-23" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">where</span>(is.character), <span class="sc">~</span> <span class="fu">case_when</span>(</span>
<span id="cb47-24"><a href="#cb47-24" aria-hidden="true" tabindex="-1"></a>      <span class="fu">str_detect</span>(., ymd_pattern) <span class="sc">~</span> <span class="fu">as.character</span>(<span class="fu">as.Date</span>(., <span class="at">format =</span> <span class="st">"%Y-%m-%d"</span>)),</span>
<span id="cb47-25"><a href="#cb47-25" aria-hidden="true" tabindex="-1"></a>      <span class="fu">str_detect</span>(., mdy_pattern) <span class="sc">~</span> <span class="fu">as.character</span>(<span class="fu">as.Date</span>(., <span class="at">format =</span> <span class="st">"%m/%d/%Y"</span>)),</span>
<span id="cb47-26"><a href="#cb47-26" aria-hidden="true" tabindex="-1"></a>      <span class="fu">str_detect</span>(., dmy_pattern) <span class="sc">~</span> <span class="fu">as.character</span>(<span class="fu">as.Date</span>(., <span class="at">format =</span> <span class="st">"%d.%m.%Y"</span>)),</span>
<span id="cb47-27"><a href="#cb47-27" aria-hidden="true" tabindex="-1"></a>      <span class="cn">TRUE</span> <span class="sc">~</span> .</span>
<span id="cb47-28"><a href="#cb47-28" aria-hidden="true" tabindex="-1"></a>    )))</span>
<span id="cb47-29"><a href="#cb47-29" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb47-30"><a href="#cb47-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-31"><a href="#cb47-31" aria-hidden="true" tabindex="-1"></a><span class="co"># Function reads excel, date columns, return tidy df</span></span>
<span id="cb47-32"><a href="#cb47-32" aria-hidden="true" tabindex="-1"></a>read_and_tidy <span class="ot">&lt;-</span> <span class="cf">function</span>(path, sheet) {</span>
<span id="cb47-33"><a href="#cb47-33" aria-hidden="true" tabindex="-1"></a>  <span class="fu">read_excel</span>(<span class="at">path =</span> path, <span class="at">sheet =</span> sheet) <span class="sc">%&gt;%</span></span>
<span id="cb47-34"><a href="#cb47-34" aria-hidden="true" tabindex="-1"></a>    <span class="fu">convert_to_date</span>() <span class="sc">%&gt;%</span></span>
<span id="cb47-35"><a href="#cb47-35" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">everything</span>(), <span class="sc">~</span> <span class="fu">str_trim</span>(.))) <span class="sc">%&gt;%</span>  </span>
<span id="cb47-36"><a href="#cb47-36" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">everything</span>(), <span class="sc">~</span> <span class="fu">str_replace_all</span>(., <span class="st">"</span><span class="sc">\\</span><span class="st">s+"</span>, <span class="st">" "</span>)))  </span>
<span id="cb47-37"><a href="#cb47-37" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb47-38"><a href="#cb47-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-39"><a href="#cb47-39" aria-hidden="true" tabindex="-1"></a><span class="co"># create dfs</span></span>
<span id="cb47-40"><a href="#cb47-40" aria-hidden="true" tabindex="-1"></a>data_list <span class="ot">&lt;-</span> <span class="fu">lapply</span>(group_sheets, read_and_tidy, <span class="at">path =</span> glucose_tracking)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>New names:
New names:
New names:
New names:
• `Week` -&gt; `Week...32`
• `Week` -&gt; `Week...34`
• `Week` -&gt; `Week...36`
• `Week` -&gt; `Week...38`</code></pre>
</div>
<div class="sourceCode cell-code" id="cb49"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb49-1"><a href="#cb49-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Combine dfs</span></span>
<span id="cb49-2"><a href="#cb49-2" aria-hidden="true" tabindex="-1"></a>tidy_data <span class="ot">&lt;-</span> <span class="fu">bind_rows</span>(data_list, <span class="at">.id =</span> <span class="st">"sheet"</span>)</span>
<span id="cb49-3"><a href="#cb49-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-4"><a href="#cb49-4" aria-hidden="true" tabindex="-1"></a><span class="co"># View data</span></span>
<span id="cb49-5"><a href="#cb49-5" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(tidy_data)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 149 × 138
   sheet `Mouse ID` Sex   Genotype DOB     `First Test Week` `8.11` `1`   `8.16`
   &lt;chr&gt; &lt;chr&gt;      &lt;chr&gt; &lt;chr&gt;    &lt;chr&gt;   &lt;chr&gt;             &lt;chr&gt;  &lt;chr&gt; &lt;chr&gt; 
 1 1     82         M     Het      6.7.23  8.11.2023         139    27.8  118   
 2 1     83         M     Het      6.7.23  8.11.2023         126    25.5  121   
 3 1     84         M     Het      6.7.23  8.11.2023         141    31.5  11    
 4 1     85         M     Wt       6.7.23  8.11.2023         177    28.9  126   
 5 1     86         M     Het      6.7.23  8.11.2023         136    28.2  121   
 6 1     87         M     Het      6.7.23  8.11.2023         125    29.9  122   
 7 1     88         F     Het      6.16.23 8.11.2023         160    23    120   
 8 1     89         F     Het      6.16.23 8.11.2023         117    22.2  104   
 9 1     90         F     Het      6.16.23 8.11.2023         138    24.7  121   
10 1     91         F     Het      6.16.23 8.11.2023         122    20.1  131   
# ℹ 139 more rows
# ℹ 129 more variables: `2` &lt;chr&gt;, `8.31.23` &lt;chr&gt;, `3` &lt;chr&gt;, `9.7.23` &lt;chr&gt;,
#   `4` &lt;chr&gt;, `9.14.23` &lt;chr&gt;, `5` &lt;chr&gt;, `9.20.23` &lt;chr&gt;, `6` &lt;chr&gt;,
#   `9.27.23` &lt;chr&gt;, `7` &lt;chr&gt;, `10.3.23` &lt;chr&gt;, `8` &lt;chr&gt;, `10.10.23` &lt;chr&gt;,
#   `9` &lt;chr&gt;, `10.17.23` &lt;chr&gt;, `10` &lt;chr&gt;, `10.24.23` &lt;chr&gt;, `11` &lt;chr&gt;,
#   `10.31.23` &lt;chr&gt;, `12` &lt;chr&gt;, `11.7.23` &lt;chr&gt;, `13` &lt;chr&gt;,
#   `11.14.23` &lt;chr&gt;, `14` &lt;chr&gt;, `11.21.23` &lt;chr&gt;, `15` &lt;chr&gt;, …</code></pre>
</div>
<div class="sourceCode cell-code" id="cb51"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb51-1"><a href="#cb51-1" aria-hidden="true" tabindex="-1"></a><span class="fu">view</span>(tidy_data)</span>
<span id="cb51-2"><a href="#cb51-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb51-3"><a href="#cb51-3" aria-hidden="true" tabindex="-1"></a>sheet5 <span class="ot">&lt;-</span> sheet_names[<span class="fu">str_detect</span>(sheet_names, <span class="fu">fixed</span>(<span class="st">"Group 5"</span>))]</span>
<span id="cb51-4"><a href="#cb51-4" aria-hidden="true" tabindex="-1"></a>sheet5_data <span class="ot">&lt;-</span> <span class="fu">lapply</span>(sheet5, read_and_tidy, <span class="at">path =</span> glucose_tracking)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>